

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Overview &mdash; St. Jude Cloud Documentation v1.0.0</title>
  

  
  
  
  

  

  
  
    

  

  
    <link rel="stylesheet" href="../../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../../_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="../../../_static/css/custom.css" type="text/css" />
    <link rel="search" title="Search" href="../../../search.html" />
    <link rel="next" title="Pipeline Operation" href="operation.html" />
    <link rel="prev" title="Rapid RNA-Seq Fusion Detection" href="index.html" /> 

  
  <script src="../../../_static/js/modernizr.min.js"></script>

</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search">
          

          
            <a href="../../../index.html" class="icon icon-home"> St. Jude Cloud Documentation
          

          
          </a>

          
            
            
              <div class="version">
                latest
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">General Guides</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../../faq.html">Frequently Asked Questions</a></li>
</ul>
<p class="caption"><span class="caption-text">Data Guides</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../data/data-request.html">Make a Data Request</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../data/fill-out-daa.html">Data Access Agreement</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../data/command-line.html">Getting Started with the Command Line</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../data/upload-data.html">Uploading Data to St. Jude Cloud</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../data/download-data.html">Downloading Data from St. Jude Cloud</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../data/data-transfer-app.html">Data Transfer Application</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../data/dac-protocols.html">Data Access Committee Policies and Protocols</a></li>
</ul>
<p class="caption"><span class="caption-text">Tool Guides</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../pecan-pie/index.html">PeCan PIE</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="index.html">Rapid RNA-Seq Fusion Detection</a><ul class="current">
<li class="toctree-l2 current"><a class="current reference internal" href="#">Overview</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#inputs">Inputs</a></li>
<li class="toctree-l3"><a class="reference internal" href="#outputs">Outputs</a></li>
<li class="toctree-l3"><a class="reference internal" href="#process">Process</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#synopsis">Synopsis</a></li>
<li class="toctree-l4"><a class="reference internal" href="#mapping">Mapping</a></li>
<li class="toctree-l4"><a class="reference internal" href="#coverage">Coverage</a></li>
<li class="toctree-l4"><a class="reference internal" href="#splice-junction-read-quantification">Splice junction read quantification</a></li>
<li class="toctree-l4"><a class="reference internal" href="#genome-wide-fusion-prediction">Genome-wide fusion prediction</a></li>
<li class="toctree-l4"><a class="reference internal" href="#low-stringency-fusion-gene-hotpot-search">Low stringency fusion gene “Hotpot” search</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="operation.html">Pipeline Operation</a></li>
<li class="toctree-l2"><a class="reference internal" href="monitoring.html">Monitoring Run Progress</a></li>
<li class="toctree-l2"><a class="reference internal" href="navigate.html">Navigating Results</a></li>
<li class="toctree-l2"><a class="reference internal" href="results.html">Analysis of Results</a></li>
<li class="toctree-l2"><a class="reference internal" href="known-issues.html">Known Issues</a></li>
<li class="toctree-l2"><a class="reference internal" href="faq.html">Frequently Asked Questions</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../chipseq/index.html">ChIP-Seq Peak Calling</a></li>
<li class="toctree-l1"><a class="reference internal" href="../warden/index.html">WARDEN Differential Expression Analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="../neoepitope/index.html">HLA Typing and Neoepitope Prediction</a></li>
</ul>
<p class="caption"><span class="caption-text">Visualization Guides</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../visualizations/bigwig.html">ProteinPaint BigWig Viewer</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../visualizations/fusion-viewer.html">ProteinPaint Fusion Viewer</a></li>
</ul>
<p class="caption"><span class="caption-text">Other Guides</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../internal/index.html">St. Jude Internal Employee Guide</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../../index.html">St. Jude Cloud Documentation</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../../index.html">Docs</a> &raquo;</li>
        
          <li><a href="index.html">Rapid RNA-Seq Fusion Detection</a> &raquo;</li>
        
      <li>Overview</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="overview">
<span id="rapid-rnaseq-overview"></span><h1>Overview<a class="headerlink" href="#overview" title="Permalink to this headline">¶</a></h1>
<div class="section" id="inputs">
<h2>Inputs<a class="headerlink" href="#inputs" title="Permalink to this headline">¶</a></h2>
<table border="1" class="docutils">
<colgroup>
<col width="15%" />
<col width="10%" />
<col width="56%" />
<col width="19%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Name</th>
<th class="head">Type</th>
<th class="head">Description</th>
<th class="head">Example</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>FastQ files</td>
<td>Input file</td>
<td>Gzipped FastQ files generated by human transcriptome sequencing</td>
<td><a href="#id1"><span class="problematic" id="id2">*</span></a>.fastq.gz or <a href="#id3"><span class="problematic" id="id4">*</span></a>.fq.gz</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="outputs">
<h2>Outputs<a class="headerlink" href="#outputs" title="Permalink to this headline">¶</a></h2>
<p>The Rapid RNA-Seq pipeline produces the following outputs:</p>
<table border="1" class="docutils">
<colgroup>
<col width="42%" />
<col width="58%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Name</th>
<th class="head">Description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Predicted gene fusions (.txt)</td>
<td>File containing putative gene fusions.</td>
</tr>
<tr class="row-odd"><td>Coverage file (.bw)</td>
<td>bigWig file containing coverage information.</td>
</tr>
<tr class="row-even"><td>Splice junction read counts (.txt)</td>
<td>Read counts for the splice junction detected.</td>
</tr>
<tr class="row-odd"><td>Interactive fusion visualization</td>
<td>Fusion visualization produced by ProteinPaint.</td>
</tr>
<tr class="row-even"><td>Interactive coverage visualization</td>
<td>Coverage visualization produced by ProteinPaint.</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="process">
<h2>Process<a class="headerlink" href="#process" title="Permalink to this headline">¶</a></h2>
<div class="section" id="synopsis">
<h3>Synopsis<a class="headerlink" href="#synopsis" title="Permalink to this headline">¶</a></h3>
<ol class="arabic simple">
<li>The raw sequence data is aligned to HG19 using standard STAR mapping.</li>
<li>A coverage bigWig (.bw) file is produced to allow the user to
assess sample quality across the genome.</li>
<li>Two gene fusion detection algorithms are run in parallel.<ul>
<li>The <strong>Fuzzion</strong> (Rice et al. unpublished data) fusion detection algorithm is run to
provide high sensitivity for recurrent gene fusions.</li>
<li>The <strong>RNAPEG</strong> (Edmonson et al. unpublished data) splice junction read counting algorithm is
run to quantify read counts for splice junctions. These splice junction
read counts are then used by <strong>Cicero</strong> (Li et al. unpublished data) to detect
putative gene fusions.</li>
</ul>
</li>
<li>Custom visualizations for putative gene fusions and genome coverage are
produced by ProteinPaint.</li>
</ol>
</div>
<div class="section" id="mapping">
<h3>Mapping<a class="headerlink" href="#mapping" title="Permalink to this headline">¶</a></h3>
<p>We use the STAR aligner to rapidly map reads to the GRCh37 human reference
genome. [PMID: 23104886]. This step generally takes around one
hour to complete assuming approximately 55-75 million paired reads are supplied.</p>
</div>
<div class="section" id="coverage">
<h3>Coverage<a class="headerlink" href="#coverage" title="Permalink to this headline">¶</a></h3>
<p>Internally developed scripts calculate the coverage of mapped reads genome wide.
The resulting bigWig file can be viewed in ProteinPaint or used for quality
control.</p>
</div>
<div class="section" id="splice-junction-read-quantification">
<h3>Splice junction read quantification<a class="headerlink" href="#splice-junction-read-quantification" title="Permalink to this headline">¶</a></h3>
<p>We use our RNAPEG software to quantify reads spanning known and novel splice
junctions. RNAPEG also corrects improper mappings at splice junction boundaries
for more accurate definition of novel splice junctions. The resulting junctions
file can be viewed along with the coverage bigWig file to gain insights into
gene expression and splicing patterns</p>
</div>
<div class="section" id="genome-wide-fusion-prediction">
<h3>Genome-wide fusion prediction<a class="headerlink" href="#genome-wide-fusion-prediction" title="Permalink to this headline">¶</a></h3>
<p>We developed an assembly-based algorithm CICERO (Clipped-reads Extended for RNA
Optimization) that is able to extend the read-length spanning fusion junctions
for detecting complex fusions. CICERO finds clipped reads and junction spanning
reads, assembles them into a contig and maps the contig back to the reference
genome. Mapped contigs are then annotated and filtered. Those with potential
genic effects including gene fusion, ITD, readthrough or circular RNA are
reported in the final_fusions.txt file. An interactive version of this file with
predictions sorted by quality can be inspected with the ProteinPaint
interactive fusion viewer.</p>
<p>An abstract describing CICERO was presented at ASHG, 2014:
<a class="reference external" href="http://www.ashg.org/2014meeting/abstracts/fulltext/f140120024.htm">http://www.ashg.org/2014meeting/abstracts/fulltext/f140120024.htm</a></p>
</div>
<div class="section" id="low-stringency-fusion-gene-hotpot-search">
<h3>Low stringency fusion gene “Hotpot” search<a class="headerlink" href="#low-stringency-fusion-gene-hotpot-search" title="Permalink to this headline">¶</a></h3>
<p>We have observed that certain fusions such as KIAA1549-BRAF in low-grade glioma
have apparently limited read support in the bam file — either due to low
expression or low tumor purity. In these cases, we use a secondary tool,
FUZZION, that performs fuzzy matching for known fusion gene junctions for every
read in the bam file (both mapped and unmapped). FUZZION can recover even a
single low quality read potentially supporting a known fusion gene junction. The
FUZZION output is a simple text file with read IDs and sequences supporting a
particular gene fusion. The fusion point is indicated with square brakets “[].”</p>
</div>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="operation.html" class="btn btn-neutral float-right" title="Pipeline Operation" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="index.html" class="btn btn-neutral" title="Rapid RNA-Seq Fusion Detection" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2016, 2017, 2018 St. Jude Developers.

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  

    <script type="text/javascript">
        var DOCUMENTATION_OPTIONS = {
            URL_ROOT:'../../../',
            VERSION:'latest',
            LANGUAGE:'None',
            COLLAPSE_INDEX:false,
            FILE_SUFFIX:'.html',
            HAS_SOURCE:  true,
            SOURCELINK_SUFFIX: '.txt'
        };
    </script>
      <script type="text/javascript" src="../../../_static/jquery.js"></script>
      <script type="text/javascript" src="../../../_static/underscore.js"></script>
      <script type="text/javascript" src="../../../_static/doctools.js"></script>
      <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

  

  
  
    <script type="text/javascript" src="../../../_static/js/theme.js"></script>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>