



<!DOCTYPE html>
<html lang="en" class="no-js">
  <head>
    <script src="//assets.adobedtm.com/launch-EN942f79c83ff84c10a45b4856a85eaccb.min.js" async></script>
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-107924368-1"></script>
    <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","UA-107924368-1")</script>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      <meta http-equiv="x-ua-compatible" content="ie=edge">
      
      
        <link rel="canonical" href="https://www.stjude.cloud/docs/guides/data/types-of-data/">
      
      
      
        <meta name="lang:clipboard.copy" content="Copy to clipboard">
      
        <meta name="lang:clipboard.copied" content="Copied to clipboard">
      
        <meta name="lang:search.language" content="en">
      
        <meta name="lang:search.pipeline.stopwords" content="True">
      
        <meta name="lang:search.pipeline.trimmer" content="True">
      
        <meta name="lang:search.result.none" content="No matching documents">
      
        <meta name="lang:search.result.one" content="1 matching document">
      
        <meta name="lang:search.result.other" content="# matching documents">
      
        <meta name="lang:search.tokenizer" content="[\s\-]+">
      
      <link rel="shortcut icon" href="../../../assets/images/favicon.ico">
      <meta name="generator" content="mkdocs-0.17.4, mkdocs-material-2.9.2">
    
    
      
        <title>Types of Data - St. Jude Cloud Documentation</title>
      
    
    
      <link rel="stylesheet" href="../../../assets/stylesheets/application.4eccddb4.css">
      
      
    
    
      <script src="../../../assets/javascripts/modernizr.20ef595d.js"></script>

      <!-- Adobe Analytics -->
      <script src="//assets.adobedtm.com/launch-EN942f79c83ff84c10a45b4856a85eaccb.min.js" async></script>
    
    
      <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
      
    
    <link rel="stylesheet" href="../../../assets/fonts/material-icons.css">
    
      <link rel="stylesheet" href="../../../css/extra.css">
    
    
  </head>
  
    <body dir="ltr">
  
    <svg class="md-svg">
      <defs>
        
        
          <svg xmlns="http://www.w3.org/2000/svg" width="416" height="448"
    viewBox="0 0 416 448" id="__github">
  <path fill="currentColor" d="M160 304q0 10-3.125 20.5t-10.75 19-18.125
        8.5-18.125-8.5-10.75-19-3.125-20.5 3.125-20.5 10.75-19 18.125-8.5
        18.125 8.5 10.75 19 3.125 20.5zM320 304q0 10-3.125 20.5t-10.75
        19-18.125 8.5-18.125-8.5-10.75-19-3.125-20.5 3.125-20.5 10.75-19
        18.125-8.5 18.125 8.5 10.75 19 3.125 20.5zM360
        304q0-30-17.25-51t-46.75-21q-10.25 0-48.75 5.25-17.75 2.75-39.25
        2.75t-39.25-2.75q-38-5.25-48.75-5.25-29.5 0-46.75 21t-17.25 51q0 22 8
        38.375t20.25 25.75 30.5 15 35 7.375 37.25 1.75h42q20.5 0
        37.25-1.75t35-7.375 30.5-15 20.25-25.75 8-38.375zM416 260q0 51.75-15.25
        82.75-9.5 19.25-26.375 33.25t-35.25 21.5-42.5 11.875-42.875 5.5-41.75
        1.125q-19.5 0-35.5-0.75t-36.875-3.125-38.125-7.5-34.25-12.875-30.25-20.25-21.5-28.75q-15.5-30.75-15.5-82.75
        0-59.25 34-99-6.75-20.5-6.75-42.5 0-29 12.75-54.5 27 0 47.5 9.875t47.25
        30.875q36.75-8.75 77.25-8.75 37 0 70 8 26.25-20.5
        46.75-30.25t47.25-9.75q12.75 25.5 12.75 54.5 0 21.75-6.75 42 34 40 34
        99.5z" />
</svg>
        
      </defs>
    </svg>
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" data-md-component="overlay" for="__drawer"></label>
    
      <a href="#data-types" tabindex="1" class="md-skip">
        Skip to content
      </a>
    
    
      <header class="md-header" data-md-component="header">
  <nav class="md-header-nav md-grid">
    <div class="md-flex">
      <div class="md-flex__cell md-flex__cell--shrink">
        <a href="https://www.stjude.cloud/docs/" title="St. Jude Cloud Documentation" class="md-header-nav__button md-logo">
          
            <img src="../../../images/stjude-logo-child-white.png" width="40" height="28">
          
        </a>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        <label class="md-icon md-icon--menu md-header-nav__button" for="__drawer"></label>
      </div>
      <div class="md-flex__cell md-flex__cell--stretch">
        <div class="md-flex__ellipsis md-header-nav__title" data-md-component="title">
          
            
              <span class="md-header-nav__topic">
                St. Jude Cloud Documentation
              </span>
              <span class="md-header-nav__topic">
                Types of Data
              </span>
            
          
        </div>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        
          
            <label class="md-icon md-icon--search md-header-nav__button" for="__search"></label>
            
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="query" data-md-state="active">
      <label class="md-icon md-search__icon" for="__search"></label>
      <button type="reset" class="md-icon md-search__icon" data-md-component="reset" tabindex="-1">
        &#xE5CD;
      </button>
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="result">
          <div class="md-search-result__meta">
            Type to start searching
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
          
        
      </div>
      
        <div class="md-flex__cell md-flex__cell--shrink">
          <div class="md-header-nav__source">
            


  


  <a href="https://github.com/stjude/sjcloud-docs/" title="Go to repository" class="md-source" data-md-source="github">
    
      <div class="md-source__icon">
        <svg viewBox="0 0 24 24" width="24" height="24">
          <use xlink:href="#__github" width="24" height="24"></use>
        </svg>
      </div>
    
    <div class="md-source__repository">
      stjude/sjcloud-docs
    </div>
  </a>

          </div>
        </div>
      
    </div>
  </nav>
</header>
    
    <div class="md-container">
      
        
      
      
      <main class="md-main">
        <div class="md-main__inner md-grid" data-md-component="container">
          
            
              <div class="md-sidebar md-sidebar--primary" data-md-component="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    <nav class="md-nav md-nav--primary" data-md-level="0">
  <label class="md-nav__title md-nav__title--site" for="__drawer">
    <span class="md-nav__button md-logo">
      
        <img src="../../../images/stjude-logo-child-white.png" width="48" height="48">
      
    </span>
    St. Jude Cloud Documentation
  </label>
  
    <div class="md-nav__source">
      


  


  <a href="https://github.com/stjude/sjcloud-docs/" title="Go to repository" class="md-source" data-md-source="github">
    
      <div class="md-source__icon">
        <svg viewBox="0 0 24 24" width="24" height="24">
          <use xlink:href="#__github" width="24" height="24"></use>
        </svg>
      </div>
    
    <div class="md-source__repository">
      stjude/sjcloud-docs
    </div>
  </a>

    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      


  <li class="md-nav__item">
    <a href="../../.." title="Welcome to St. Jude Cloud" class="md-nav__link">
      Welcome to St. Jude Cloud
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../../../create-an-account/" title="Create An Account" class="md-nav__link">
      Create An Account
    </a>
  </li>

    
      
      
      

  


  <li class="md-nav__item md-nav__item--active md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-3" type="checkbox" id="nav-3" checked>
    
    <label class="md-nav__link" for="nav-3">
      Data Guides
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-3">
        Data Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          

  


  <li class="md-nav__item md-nav__item--active">
    
    <input class="md-toggle md-nav__toggle" data-md-toggle="toc" type="checkbox" id="__toc">
    
      
    
    
      <label class="md-nav__link md-nav__link--active" for="__toc">
        Types of Data
      </label>
    
    <a href="./" title="Types of Data" class="md-nav__link md-nav__link--active">
      Types of Data
    </a>
    
      
<nav class="md-nav md-nav--secondary">
  
  
    
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#bam-files" title="BAM files" class="md-nav__link">
    BAM files
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#whole-genome-and-whole-exome" title="Whole Genome and Whole Exome" class="md-nav__link">
    Whole Genome and Whole Exome
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rna-seq" title="RNA-Seq" class="md-nav__link">
    RNA-Seq
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gvcf-files" title="gVCF files" class="md-nav__link">
    gVCF files
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#somatic-vcf-files" title="Somatic VCF files" class="md-nav__link">
    Somatic VCF files
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cnv-files" title="CNV files" class="md-nav__link">
    CNV files
  </a>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
    
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../metadata/" title="Metadata Provided" class="md-nav__link">
      Metadata Provided
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../data-request/" title="Making a Data Request" class="md-nav__link">
      Making a Data Request
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../working-with-our-data/" title="Working with Our Data" class="md-nav__link">
      Working with Our Data
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../creating-a-cloud-app/" title="Creating a Cloud App" class="md-nav__link">
      Creating a Cloud App
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../data-transfer-app/" title="Data Transfer Application" class="md-nav__link">
      Data Transfer Application
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../command-line/" title="Command Line Interaction" class="md-nav__link">
      Command Line Interaction
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-4" type="checkbox" id="nav-4">
    
    <label class="md-nav__link" for="nav-4">
      Tool Guides
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-4">
        Tool Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../../tools/pecan-pie/" title="Pecan PIE" class="md-nav__link">
      Pecan PIE
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../tools/neoepitope/" title="NeoepitopePred" class="md-nav__link">
      NeoepitopePred
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../tools/chipseq/" title="ChIP-Seq Peak Calling" class="md-nav__link">
      ChIP-Seq Peak Calling
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../tools/rapid-rnaseq/" title="Rapid RNA-Seq Fusion Detection" class="md-nav__link">
      Rapid RNA-Seq Fusion Detection
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../tools/warden/" title="WARDEN Differential Expression Analysis" class="md-nav__link">
      WARDEN Differential Expression Analysis
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../tools/mutational-signatures/" title="Mutational Signatures" class="md-nav__link">
      Mutational Signatures
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../tools/cis-x/" title="" class="md-nav__link">
      
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-5" type="checkbox" id="nav-5">
    
    <label class="md-nav__link" for="nav-5">
      Visualization Guides
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-5">
        Visualization Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../../portals/pecan/" title="PeCan Data Portal" class="md-nav__link">
      PeCan Data Portal
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../portals/genome-paint/" title="Genome Paint" class="md-nav__link">
      Genome Paint
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../portals/sickle-cell/" title="Sickle Cell Genomics Portal" class="md-nav__link">
      Sickle Cell Genomics Portal
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-6" type="checkbox" id="nav-6">
    
    <label class="md-nav__link" for="nav-6">
      Forms
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-6">
        Forms
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../../forms/how-to-fill-out-DAA/" title="Filling out a Data Access Agreement" class="md-nav__link">
      Filling out a Data Access Agreement
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../forms/how-to-fill-out-Extension/" title="Filling out an Extension Addendum" class="md-nav__link">
      Filling out an Extension Addendum
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../../../citation-reference/" title="Citing St. Jude Cloud" class="md-nav__link">
      Citing St. Jude Cloud
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../../../faq/" title="Frequently Asked Questions" class="md-nav__link">
      Frequently Asked Questions
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../../../glossary/" title="Glossary" class="md-nav__link">
      Glossary
    </a>
  </li>

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              <div class="md-sidebar md-sidebar--secondary" data-md-component="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary">
  
  
    
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#bam-files" title="BAM files" class="md-nav__link">
    BAM files
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#whole-genome-and-whole-exome" title="Whole Genome and Whole Exome" class="md-nav__link">
    Whole Genome and Whole Exome
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rna-seq" title="RNA-Seq" class="md-nav__link">
    RNA-Seq
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gvcf-files" title="gVCF files" class="md-nav__link">
    gVCF files
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#somatic-vcf-files" title="Somatic VCF files" class="md-nav__link">
    Somatic VCF files
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cnv-files" title="CNV files" class="md-nav__link">
    CNV files
  </a>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content">
            <article class="md-content__inner md-typeset">
              
                
                  <a href="https://github.com/stjude/sjcloud-docs/edit/master/docs/guides/data/types-of-data.md" title="Edit this page" class="md-icon md-content__icon">&#xE3C9;</a>
                
                
                <h1 id="data-types">Data types</h1>
<p>St. Jude Cloud hosts both raw genomic data files and processed results files:</p>
<table>
<thead>
<tr>
<th>File Type</th>
<th>Short Description</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td>BAM</td>
<td>HG38 aligned BAM files produced by <a href="https://azure.microsoft.com/en-us/services/genomics/">Microsoft Genomics Service</a> (DNA-Seq) or STAR 2-pass mapping (RNA-Seq).</td>
<td><a href="#bam-files">Click here</a></td>
</tr>
<tr>
<td>gVCF</td>
<td><a href="https://software.broadinstitute.org/gatk/documentation/article?id=11004">Genomic VCF</a> files produced by <a href="https://azure.microsoft.com/en-us/services/genomics/">Microsoft Genomics Service</a>.</td>
<td><a href="#gvcf-files">Click here</a></td>
</tr>
<tr>
<td>Somatic VCF</td>
<td>Curated list of somatic variants produced by the St. Jude somatic variant analysis pipeline.</td>
<td><a href="#somatic-vcf-files">Click here</a></td>
</tr>
<tr>
<td>CNV</td>
<td>list of somatic copy number alterations produced by St. Jude CONSERTING pipeline.</td>
<td><a href="#cnv-files">Click here</a></td>
</tr>
</tbody>
</table>
<h4 id="bam-files">BAM files</h4>
<p>In St. Jude Cloud, we store aligned sequence reads in BAM file format for whole genome sequencing, whole exome sequencing, and RNA-seq. (For more information on SAM/BAM files, please refer to the <a href="https://samtools.github.io/hts-specs/SAMv1.pdf">SAM/BAM specification</a>.) For research samples, we require the standard 30X coverage for whole genome and 100X for whole exome sequencing. For clinical samples, we require higher coverage, 45X, for whole genome sequencing due to tumor purity issues found in clinical tumor specimens. For RNA-Seq, since only a subset of genes are expressed in a specific tissue, we require 30% of the exons to have 20X coverage in order to ensure that at least 30% of the expressed genes have sufficient coverage.</p>
<h5 id="whole-genome-and-whole-exome">Whole Genome and Whole Exome</h5>
<p>Whole Genome Sequence (WGS) and Whole Exome Sequence (WES) BAM files were produced by the <a href="https://azure.microsoft.com/en-us/services/genomics/">Microsoft Genomics Service</a> aligned to HG38 (GRCh38 no alt analysis set). For more information about how Microsoft Genomics produces BAM files or any other questions regarding data generation, please refer to <a href="https://azure.microsoft.com/en-us/resources/accelerate-precision-medicine-with-microsoft-genomics/">the official Microsoft Genomics whitepaper</a>.</p>
<h5 id="rna-seq">RNA-Seq</h5>
<p>RNA-Seq BAM files are mapped to HG38 + <a href="https://www.thermofisher.com/order/catalog/product/4456740">ERCC Spike In Sequences</a> (commonly used for normalization of expression analyses). For alignment, <code class="codehilite"><span class="n">STAR</span></code> v2.5.3a 2-pass mapping followed by <code class="codehilite"><span class="n">Picard</span> <span class="n">MarkDuplicates</span></code>. Below is the <code class="codehilite"><span class="n">STAR</span></code> command used during alignment. For more information about any of the parameters used, please refer to the <a href="https://github.com/alexdobin/STAR/blob/7283af27e84839e93ecf7ed6a14c8ff675fdf79c/doc/STARmanual.pdf">STAR manual</a> for v2.5.3a.</p>
<div class="codehilite"><pre><span></span>    STAR <span class="se">\</span>
        --runThreadN <span class="nv">$NUM_THREADS</span> <span class="se">\ </span>                          <span class="c1"># $NUM_THREADS is the number of threads to parallelize the alignment across (generally we use 4).</span>
        --genomeDir <span class="nv">$GENOME_DIR</span> <span class="se">\ </span>                            <span class="c1"># $GENOME_DIR is a STAR reference directory containing HG38 and ERCC Spike In sequences.</span>
        --readFilesIn <span class="nv">$READ_FILES</span> <span class="se">\ </span>                          <span class="c1"># $READ_FILES are the input FastQ files to align.</span>
        --limitBAMsortRAM <span class="nv">$MEMORY_LIMIT</span> <span class="se">\ </span>                    <span class="c1"># $MEMORY_LIMIT is a upper limit on the amount of RAM to use in the alignment.</span>
        --outFileNamePrefix <span class="nv">$OUT_FILE_PREFIX</span> <span class="se">\</span>
        --outSAMtype BAM SortedByCoordinate <span class="se">\</span>
        --outSAMstrandField intronMotif <span class="se">\</span>
        --outSAMattributes NH   HI   AS   nM   NM   MD   XS <span class="se">\</span>
        --outSAMunmapped Within <span class="se">\</span>
        --outSAMattrRGline <span class="nv">$RGs</span> <span class="se">\ </span>                            <span class="c1"># $RGs is the read group information for each FastQ passed in $READ_FILES.</span>
        --outFilterMultimapNmax <span class="m">20</span> <span class="se">\</span>
        --outFilterMultimapScoreRange <span class="m">1</span> <span class="se">\</span>
        --outFilterScoreMinOverLread <span class="m">0</span>.66 <span class="se">\</span>
        --outFilterMatchNminOverLread <span class="m">0</span>.66 <span class="se">\</span>
        --outFilterMismatchNmax <span class="m">10</span> <span class="se">\</span>
        --alignIntronMax <span class="m">500000</span> <span class="se">\</span>
        --alignMatesGapMax <span class="m">1000000</span> <span class="se">\</span>
        --alignSJDBoverhangMin <span class="m">1</span> <span class="se">\</span>
        --sjdbScore <span class="m">2</span> <span class="se">\</span>
        --twopassMode Basic
</pre></div>


<h4 id="gvcf-files">gVCF files</h4>
<p>We provide gVCF files produced by the <a href="https://azure.microsoft.com/en-us/services/genomics/">Microsoft Genomics Service</a>. gVCF files are derived from the BAM files produced above as called by <a href="https://software.broadinstitute.org/gatk/documentation/tooldocs/3.8-0/org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeCaller.php">GATK's haplotype caller</a>. Today, we defer to <a href="https://software.broadinstitute.org/gatk/documentation/article?id=11004">the official specification document</a> from the Broad Institute, as well as <a href="https://software.broadinstitute.org/gatk/documentation/article?id=4017">this discussion</a> on the difference between VCF and gVCF files. For more information about how Microsoft Genomics produces gVCF files or any other questions regarding data generation, please refer to <a href="https://azure.microsoft.com/en-us/resources/accelerate-precision-medicine-with-microsoft-genomics/">the official Microsoft Genomics whitepaper</a>.</p>
<h4 id="somatic-vcf-files">Somatic VCF files</h4>
<p>Somatic VCF files contain HG38 based SNV/Indel variant calls from the St. Jude somatic variant analysis pipeline as follows. Broadly speaking:</p>
<ol>
<li>Reads were aligned to HG19 using <a href="http://bio-bwa.sourceforge.net/">bwa backtrack</a> (<code class="codehilite"><span class="n">bwa</span> <span class="n">aln</span></code> + <code class="codehilite"><span class="n">bwa</span> <span class="n">sampe</span></code>) using default parameters.</li>
<li>Post processing of aligned reads was performed using <a href="https://broadinstitute.github.io/picard/">Picard</a> <code class="codehilite"><span class="n">CleanSam</span></code> and <code class="codehilite"><span class="n">MarkDuplicates</span></code>.</li>
<li>Variants were called using the <a href="https://academic.oup.com/bioinformatics/article/27/6/865/236751">Bambino</a> variant caller (you can download by navigating <a href="https://www.stjuderesearch.org/site/lab/zhang">here</a> and searching for "Bambino package").</li>
<li>Variants were post-processed using an in-house post-processing pipeline that cleans and annotates variants. This pipeline is not currently publicly available.</li>
<li>Variants were manually reviewed by analysts and published with <a href="https://www.stjude.org/research/pediatric-cancer-genome-project.html">the relevant Pediatric Cancer Genome Project (PCGP) paper</a>.</li>
<li>Post-publication, variants were lifted over to HG38 (the original HG19 coordinates are stored in the <code class="codehilite"><span class="n">HG19</span></code> INFO field.).</li>
</ol>
<p>For more information on variants for each of the individuals, please refer to the relevant PCGP paper. For more information on the variant calling format (VCF), please see the latest specification for VCF document listed <a href="https://samtools.github.io/hts-specs/">here</a>.</p>
<h4 id="cnv-files">CNV files</h4>
<p>CNV files contain copy number alteration (CNA) analysis results for paired tumor-normal WGS samples. Files are produced by running paired tumor-normal BAM files through the <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4591043/">CONSERTING</a> pipeline which identifies CNA through iterative analysis of (i) local segmentation by read depth within boundaries identified by structural variation (SV) breakpoints followed by (ii) segment merging and local SV analysis. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3527068/">CREST</a> was used to identify local SV breakpoints. CNV files contain the following information:</p>
<table>
<thead>
<tr>
<th>Field</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>chrom</td>
<td>chromosome</td>
</tr>
<tr>
<td>loc.start</td>
<td>start of segment</td>
</tr>
<tr>
<td>loc.end</td>
<td>end of segment</td>
</tr>
<tr>
<td>num.mark</td>
<td>number of windows retained in the segment (gaps and windows with low mappability are excluded)</td>
</tr>
<tr>
<td>length.ratio</td>
<td>The ratio between the length of the used windows to the genomic length</td>
</tr>
<tr>
<td>seg.mean</td>
<td>The estimated GC corrected difference signal (2 copy gain will have a seg.mean of 1)</td>
</tr>
<tr>
<td>GMean</td>
<td>The mean coverage in the germline sample (a value of 1 represents diploid)</td>
</tr>
<tr>
<td>DMean</td>
<td>The mean coverage in the tumor sample</td>
</tr>
<tr>
<td>LogRatio</td>
<td>Log2 ratio between tumor and normal coverage</td>
</tr>
<tr>
<td>Quality score</td>
<td>A empirical score used in merging</td>
</tr>
<tr>
<td>SV_Matching</td>
<td>whether the boundary of the segments were supported by SVs (3: both ends supported, 2: right end supported, 1: left end supported, 0: neither end supported)</td>
</tr>
</tbody>
</table>
                
                  
                
              
              
                


              
            </article>
          </div>
        </div>
      </main>
      
        
<footer class="md-footer">
  
    <div class="md-footer-nav">
      <nav class="md-footer-nav__inner md-grid">
        
          <a href="../../../create-an-account/" title="Create An Account" class="md-flex md-footer-nav__link md-footer-nav__link--prev" rel="prev">
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-back md-footer-nav__button"></i>
            </div>
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Previous
                </span>
                Create An Account
              </span>
            </div>
          </a>
        
        
          <a href="../metadata/" title="Metadata Provided" class="md-flex md-footer-nav__link md-footer-nav__link--next" rel="next">
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Next
                </span>
                Metadata Provided
              </span>
            </div>
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-forward md-footer-nav__button"></i>
            </div>
          </a>
        
      </nav>
    </div>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
          <div class="md-footer-copyright__highlight">
            Copyright &copy; 2017 - 2019 St. Jude Children's Research Hospital
          </div>
        
        
        built from <a href="https://github.com/stjude/sjcloud-docs/tree/c16b33d">c16b33d</a> using
        
        <a href="https://www.mkdocs.org">MkDocs</a>
        and a modified version of
        <a href="https://squidfunk.github.io/mkdocs-material/">
          Material for MkDocs</a>.
      </div>
      
        
  <div class="md-footer-social">
    <link rel="stylesheet" href="../../../assets/fonts/font-awesome.css">
    
      <a href="https://www.facebook.com/stjuderesearch/" class="md-footer-social__link fa fa-facebook"></a>
    
      <a href="https://twitter.com/stjuderesearch?lang=en" class="md-footer-social__link fa fa-twitter"></a>
    
      <a href="https://www.linkedin.com/company/st-jude-childrens-research-hospital" class="md-footer-social__link fa fa-linkedin"></a>
    
      <a href="https://github.com/stjude" class="md-footer-social__link fa fa-github"></a>
    
  </div>

      
    </div>
  </div>
</footer>
      
    </div>
    
      <script src="../../../assets/javascripts/application.ec5be3af.js"></script>
      
      <script>app.initialize({version:"0.17.4",url:{base:"../../.."}})</script>
      
    
    
      
    
  </body>
</html>