Overview
====================

Inputs
------------------------------------------------------------------------------------------------------------------------------------------------------------
The HLA typing and neoepitope prediction workflow two required inputs: 
 1. FastQs or BAM file and BAM index 
 2. Mutation file

+-------------------+------------+----------------------------------------------------------------+---------------------+
| Name              | Type       | Description                                                    | Example             |  
+===================+============+================================================================+=====================+
| FastQ files       | Input file | Gzipped FastQ files generated by experiment                    | *.fastq or *.fq     |
+-------------------+------------+----------------------------------------------------------------+---------------------+
| BAM files         | Input file | BAM files aligned against Hg19/Hg38 (WGS, WES or RNAseq)       | *.bam               |
+-------------------+------------+----------------------------------------------------------------+---------------------+
| BAM index         | Input file | BAM index of the BAM file                                      | *.bai               |
+-------------------+------------+----------------------------------------------------------------+---------------------+
| Mutation file     | Input file | Mutation information                                           | *.txt(tab-delimited)|
+-------------------+------------+----------------------------------------------------------------+---------------------+
| SNV or fusion     | Parameter  | Specify the mutation file contains SNV or gene fusion          | SNV                 |
+-------------------+------------+----------------------------------------------------------------+---------------------+
| Peptide size      | Parameter  | Size of the peptide                                            | 9                   |
+-------------------+------------+----------------------------------------------------------------+---------------------+
| Affinity threshold| Parameter  | Affinity cutoff for epitope prediction report                  | 500                 |
+-------------------+------------+----------------------------------------------------------------+---------------------+

Outputs
-------

+---------------------------------------+-----------------------------------------------------------------------------------+
| Name                                  | Description                                                                       |
+=======================================+===================================================================================+
| Epitope affinity prediction (html)    | Epitope affinity. The peptide with affinity < cutoff will be highlighted.         |
+---------------------------------------+-----------------------------------------------------------------------------------+
| Epitope affinity prediction (xlsx)    | Excel tables for the infomation of all epitopes                                   |
+---------------------------------------+-----------------------------------------------------------------------------------+
| Affinity (raw output)                 | Epitope affinity                                                                  |
+---------------------------------------+-----------------------------------------------------------------------------------+
| Peptide sequence (raw output)         | Peptide sequences in fasta format                                                 |
+---------------------------------------+-----------------------------------------------------------------------------------+

Theory
--------------------

HLA Typing Algorithm
++++++++++++++++++++

The HLA typing algorithm is used to predict the HLA class I alleles. Users
can either provide fastq (paired or single end reads) or a BAM file as input.
When using a BAM file as input, the reads surrounding the HLA loci and unmapped
reads will be extracted. The reads will be fed into Optitype for HLA typing.
The default settings for Optitype are used. The output of the HLA type can be
combined with the our epitope detection algorithm to perform affinity
prediction of neoepitopes (see :ref:`epitope-pred`).
 
.. _epitope-pred:

Epitope Prediction Algorithm
++++++++++++++++++++++++++++

The stjude-epitope applet is used to extract peptides covering
an array of tiling peptides (size defined by users) overlapping each missense
mutation or gene fusion. Fusion junctions can be identified using RNAseq by
fusion detection tools (e.g. CICERO, Li et al.). NetMHCcons (Karosiene et al.) is
subsequently used to predict affinities of the peptide array for each HLA
receptor in each sample. The neoepitope with affinity lower than the threshold
will be highlighted in output file (default 500 nM).

Process
-------

HLA typing:
 If you use **FastQ** files as input:
  1. The input fastqs will be aligned against the Optitype HLA reference sequences using razers3 (see https://github.com/FRED-2/OptiType).
  2. The fished fastqs will be used for HLA typing using Opitype.

 If you use **BAM** files as input:
  1. The reads falling within the HLA loci and their paralogous loci will be extracted. 
  2. The reads unmapped to the human genome will be extracted.
  3. The reads from step 1 and 2 will be combined and deduplicated (in fastq format).
  4. The input fastqs will be aligned against the Optitype HLA reference sequences using razers3 (see https://github.com/FRED-2/OptiType).
  5. The fished fastqs will be used for HLA typing using Opitype.

Neoepitope prediction:
 1. Check the version of the genomic position of the input SNV/fusion file.
 2. lift over the genomic coordinations if the reference genomic position is not Hg19. Currently, the internal genome annotation was based on Hg19 and the genome coordiantes of the mutation files will be adjusted to Hg19 for peptide extraction.
 3. Extract the peptide flanking the mutations.
 4. Run NetMHCcons to obtain the affinity prediction of the peptides.
 5. Produce the affinity report of each peptide.

**References**

 1. Szolek A, Schubert B, Mohr C, Sturm M, Feldhahn M, Kohlbacher O: OptiType: precision HLA typing from next-generation sequencing data. Bioinformatics 2014, 30:3310-3316.
 2. Karosiene E, Lundegaard C, Lund O, Nielsen M: NetMHCcons: a consensus method for the major histocompatibility complex class I predictions. Immunogenetics 2012, 64:177-186.