|                       |                                            |
| --------------------- | ------------------------------------------ |
| **Authors**           | Scott Newman, Clay McLeod, Yongjin Li      |
| **Publication**       | N/A (not published)                        |
| **Technical Support** | [Contact Us](https://stjude.cloud/contact) |

## Introduction

Fusion genes are important for cancer diagnosis, subtype definition and
targeted therapy. RNASeq is useful for detecting fusion transcripts.
Computational methods face challenges to identify fusion transcripts
arising from internal tandem duplication (ITD), multiple genes, low
expression or non-templated insertions. Here we present an end-to-end
clinically validated pipeline "Rapid RNA-Seq" that detects gene fusions
and ITDs from human RNA-Seq.

## Overview

### Inputs

The input can be either of the two entries below based on whether you want to start
with FastQ files or a BAM file.

| Name | Description | Example |
|--|--|--|
| Paired FastQ files | Gzipped FastQ files generated by human RNA-Seq | Sample_R1.fastq.gz and Sample_R2.fastq.gz |
| BAM file | Aligned reads file from human RNA-Seq | Sample.bam |

!!! note
    If you provide a BAM file to the pipeline, it **must** be aligned to GRCh37-lite.
    Running a BAM aligned to any other reference genome is not supported. Maybe more
    importantly, we do not check the genome build of the BAM, so errors in computation
    or the results can occur. If your BAM is *not* aligned to this genome build, we 
    recommend converting the BAM back to FastQ files using 
    [Picard's SamToFastq](https://broadinstitute.github.io/picard/command-line-overview.html#SamToFastq)
    functionality and using the FastQ version of the pipeline.

### Outputs

The Rapid RNA-Seq pipeline produces the following outputs:

| Name | Description |
|--|--|
| Predicted gene fusions (.txt) | File containing putative gene fusions. |
| Coverage file (.bw) | bigWig file containing coverage information. |
| Splice junction read counts (.txt) | Read counts for the splice junction detected. |
| Interactive fusion visualization | Fusion visualization produced by ProteinPaint. |
| Interactive coverage visualization | Coverage visualization produced by ProteinPaint. |

### Process

1. The raw sequence data is aligned to GRCh37-lite using standard STAR
   mapping.
2. A coverage bigWig (.bw) file is produced to allow the user to assess
   sample quality across the genome.
3. Two gene fusion detection algorithms are run in parallel.
    * The **Fuzzion** (Rice et al. unpublished data) fusion detection
      algorithm is run to provide high sensitivity for recurrent gene
      fusions.
    * The **RNAPEG** (Edmonson et al. unpublished data) splice
      junction read counting algorithm is run to quantify read counts
      for splice junctions. These splice junction read counts are then
      used by **Cicero** (Li et al. unpublished data) to detect
      putative gene fusions.
4. Custom visualizations for putative gene fusions and genome coverage
   are produced by ProteinPaint.

**Mapping**

We use the [STAR aligner](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905/) 
to rapidly map reads to the GRCh37 human reference genome. This step generally 
takes around one hour to complete assuming approximately 55-75 million paired reads
are supplied.

**Coverage**

Internally developed scripts calculate the coverage of mapped reads
genome wide. The resulting bigWig file can be viewed in ProteinPaint or
used for quality control.

**Splice junction read quantification**

We use our RNAPEG software to quantify reads spanning known and novel
splice junctions. RNAPEG also corrects improper mappings at splice
junction boundaries for more accurate definition of novel splice
junctions. The resulting junctions file can be viewed along with the
coverage bigWig file to gain insights into gene expression and splicing
patterns

**Genome-wide fusion prediction**

We developed an assembly-based algorithm CICERO (Clipped-reads Extended
for RNA Optimization) that is able to extend the read-length spanning
fusion junctions for detecting complex fusions. CICERO finds clipped
reads and junction spanning reads, assembles them into a contig and maps
the contig back to the reference genome. Mapped contigs are then
annotated and filtered. Those with potential genic effects including
gene fusion, ITD, readthrough or circular RNA are reported in the
`final_fusions.txt` file. An interactive version of this file with
predictions sorted by quality can be inspected with the ProteinPaint
interactive fusion viewer.

An abstract describing CICERO was presented at ASHG, 2014:
<http://www.ashg.org/2014meeting/abstracts/fulltext/f140120024.htm>

**Low stringency fusion gene "Hotpot" search**

We have observed that certain fusions such as KIAA1549-BRAF in low-grade
glioma have apparently limited read support in the bam file â€” either due
to low expression or low tumor purity. In these cases, we use a
secondary tool, FUZZION, that performs fuzzy matching for known fusion
gene junctions for every read in the bam file (both mapped and
unmapped). FUZZION can recover even a single low quality read
potentially supporting a known fusion gene junction. The FUZZION output
is a simple text file with read IDs and sequences supporting a
particular gene fusion. The fusion point is indicated with square
brackets `[]`.

## Pipeline Operation

!!! caution
    This pipeline assumes GRCh37-lite coordinates. If your BAM is 
    *not* aligned to this genome build, we recommend converting the BAM 
    back to FastQ files using [Picard's SamToFastq](https://broadinstitute.github.io/picard/command-line-overview.html#SamToFastq)
    functionality.

## Getting started

To get started, you need to navigate to the [Rapid RNA-Seq tool page](https://platform.stjude.cloud/tools/rapid_rna-seq). You'll need to click
the "Start" button in the left hand pane. This creates a cloud workspace
in DNAnexus with the same name as the tool. After this, you will be able 
to upload your input files to that workspace.

![](../../images/guides/tools/rapid-rnaseq/click-start.gif)

!!! note
    If you can't see the "Start" button, one of these two scenarios is likely the case:

    * You see three buttons on the left sidebar instead of one. In this case,
      you've already clicked the "Start" button previously, and a cloud workspace has
      already been created for you. In this case, you're good! You can move
      on to the next section.
    * If you cannot see *any* buttons on the left side, you probably have not
      logged in yet. If you see a sentence that says "Log in to launch this 
      tool", simply login and try again.

    If neither of these are the case and you still can't click "Start",
    [contact us](https://stjude.cloud/contact).

## Uploading data

The Rapid RNA-Seq pipeline takes either a paired set of Gzipped FastQ files or 
a GRCh37-lite aligned BAM from human RNA-Seq. You can upload your input file(s)
using the [data transfer application](../data/data-transfer-app.md)
or by uploading them through [the command line](../data/command-line.md).
Both of the guides linked here will contain more details on how to upload
data using that method, so we defer to those guides here.

!!! tip
    If you plan to upload data through the St. Jude Cloud Data Transfer application
    (recommended), you can click the "Upload Data" button in the left panel. If you
    have not already downloaded the app, do so by clicking "Download app". Once you
    have the app, you can click "Open app" to open the app with the tool's cloud 
    workspace already opened and ready to drag-and-drop files into it!

    For more information, check out the [data transfer application](../data/data-transfer-app.md) guide.

## Running the tool

Once you've uploaded data to your cloud workspace, 
click "Launch Tool" on the [tool's landing page](https://platform.stjude.cloud/tools/rapid_rna-seq). 
A dropdown will present the different presets for running the workflow.
Here, you can select whether you wish to start with FastQ files or a BAM file.

![](../../images/guides/tools/rapid-rnaseq/launch-tool.gif)

!!! todo
    Continue converting guide.

1.  Click the `Fastq/R1` input file slot in orange.

    ![image](../../images/guides/tools/rapid-rnaseq//rrs-startup.png)

2.  Select the "read 1" Gzipped FastQ file from your workspace.

    ![image](../../images/guides/tools/rapid-rnaseq//rrs-r1-file.png)

3.  Click the `Fastq/R2` input file slot in orange.

    ![image](../../images/guides/tools/rapid-rnaseq//rrs-intermediate-input.png)

4.  Select the "read 2" Gzipped FastQ file from your workspace.

    ![image](../../images/guides/tools/rapid-rnaseq//rrs-r2-file.png)

### Parameter Configuration

There are no parameters to configure for the Rapid RNA-Seq pipeline.

### Running the Pipeline

You should now see a screen similar to the one below. Note that the
green "Run as Analysis" button in the top-right is no longer disabled.
You can click the button to start the analysis.

![image](../../images/guides/tools/rapid-rnaseq//run-as-analysis.png)

You should now see a screen similar to the one below.

![image](../../images/guides/tools/rapid-rnaseq//monitor.png)

You can continue on to the next section to learn about monitoring your
jobs.

## Monitoring Run Progress

!!! note
    Monitoring the progress of your runs is the same for all pipelines. This
    guide will feature the <span role="ref">rapid-rnaseq</span> pipeline,
    but you can follow along for any tool.

Monitoring the status of your pipelines in the St. Jude Cloud is simple.
First, navigate to your tool's description page (for instance, Rapid
RNA-Seq's description page is
[here](https://platform.stjude.cloud/tools/rapid_rna-seq)). You should
see a screen similar to the one in the screenshot below. In the left
hand pane, select "View Running Jobs".

![Click "View Running Jobs"](../../images/guides/tools/common/running-jobs.png)

Here, you will see a list of all of your previous runs for a tool, as
well as the job status (successful, failed, terminated by user) and
cost. You can click on the "+" icon to the left of each tool run to see
all of the different tools that were run.

![Click the "+" beside a run to view more information.](../../images/guides/tools/common/jobs-basic.png)

Other information, such as time, cost of individual job, and even job
logs are available by clicking on the sub-items.

![Click on individual items in the list to explore detailed information
on each analysis that was run.](../../images/guides/tools/common/jobs-advanced.png)

!!! note
    Advanced users can view the [DNAnexus Job Monitoring
    Tutorial](https://wiki.dnanexus.com/UI/Jobs) and the [DNAnexus Command
    Line Tutorial for Job
    Monitoring](https://wiki.dnanexus.com/Command-Line-Client/Monitoring-and-Listing-Jobs)
    for more information.

## Navigating Results

!!! note
    Navigating to the raw results of your runs is the same for all
    pipelines. This guide will feature the <span
    role="ref">rapid-rnaseq</span> pipeline, but you can follow along for
    any tool.

### Raw result files

Navigate to your tool's description page (for instance, Rapid RNA-Seq's
description page is
[here](https://platform.stjude.cloud/tools/rapid_rna-seq)). You should
see a screen similar to the one in the screenshot below. In the left
hand pane, select "View Results Files".

![Click "View Results Files"](../../images/guides/tools/common/raw-results.png)

You should now be in the tool's workspace with access to files that you
uploaded and results files that are generated. How/where the result
files are generated are specific to each pipeline. Please refer to your
individual pipeline's documentation on where the output files are kept.

![image](../../images/guides/tools/common/raw-results-success.png)

### Custom visualization results

Navigate to your tool's description page (for instance, Rapid RNA-Seq's
description page is
[here](https://platform.stjude.cloud/tools/rapid_rna-seq)). You should
see a screen similar to the one in the screenshot below. In the left
hand pane, select "Visualize Results".

![Click "Visualize Results"](../../images/guides/tools/common/visualize-results.png)

You should now be in the tool's workspace with access to files that you
uploaded and results files that are generated. How/where the result
files are generated are specific to each pipeline. Please refer to your
individual pipeline's documentation on where the output files are kept.

![image](../../images/guides/tools/common/visualize-results-success.png)

## Analysis of Results

The complete output file specification is listed on the Rapid RNA-Seq
<span role="ref">rapid-rnaseq-overview</span> page. Here, we will
discuss the different output files and how you can use them.

### Raw Result Files

-   **Predicted gene fusions**: The predicted gene fusions will be
    output with the filename `SAMPLE.final_fusions.txt`. This file is a
    tab-delimited file containing many fields for each of the predicted
    SV. The most important columns are the following.

    <table style="width:89%;">
    <colgroup>
    <col style="width: 31%" />
    <col style="width: 56%" />
    </colgroup>
    <thead>
    <tr class="header">
    <th>Field Name</th>
    <th>Description</th>
    </tr>
    </thead>
    <tbody>
    <tr class="odd">
    <td><code class="sourceCode">sample</code></td>
    <td>Sample name</td>
    </tr>
    <tr class="even">
    <td><code class="sourceCode">gene*</code></td>
    <td>Gene name</td>
    </tr>
    <tr class="odd">
    <td><code class="sourceCode">chr*</code></td>
    <td>Chromosome name</td>
    </tr>
    <tr class="even">
    <td><code class="sourceCode">pos*</code></td>
    <td>Genomic Locaton</td>
    </tr>
    <tr class="odd">
    <td><code class="sourceCode">ort*</code></td>
    <td>Strand</td>
    </tr>
    <tr class="even">
    <td>:code:reads*</td>
    <td>Supporting reads</td>
    </tr>
    <tr class="odd">
    <td><code class="sourceCode">medal</code></td>
    <td>Estimated pathogenicity assessment</td>
    </tr>
    </tbody>
    </table>

-   **Coverage file**: A standard
    [bigWig](https://genome.ucsc.edu/goldenpath/help/bigWig.html) file
    used to describe genomic coverage data.
-   **Splice junction read counts**: A custom file format describing the
    junction read counts. The following fields are included in the
    tab-delimited output file.

    <table>
    <colgroup>
    <col style="width: 4%" />
    <col style="width: 95%" />
    </colgroup>
    <thead>
    <tr class="header">
    <th>Name</th>
    <th>Description</th>
    </tr>
    </thead>
    <tbody>
    <tr class="odd">
    <td>junction</td>
    <td>Splice junction in the TCGA format &quot;chrX:start:+,chrX:end,+&quot;. &quot;start&quot; and &quot;end&quot; are the 1-based position of the last mapped nucleotide before the skip and the first mapped nucleotide after the skip (i.e. the last base of the previous exon and the first base of the next exon). Note that in .bed output these coordinates will be different, see the .bed output section below. The &quot;+&quot; is currently hardcoded, though this may change in the future.</td>
    </tr>
    <tr class="even">
    <td>count</td>
    <td>Raw count of reads supporting the junction. During correction counts for ambiguous junctions can be combined, though obviously these additional reads will not be visible in the raw BAM file.</td>
    </tr>
    <tr class="odd">
    <td>type</td>
    <td>either &quot;known&quot; (matching a reference junction) or &quot;novel&quot; (not observed in the reference junction collection).</td>
    </tr>
    <tr class="even">
    <td>genes</td>
    <td>gene symbols from the junction calling process. These still need work in the raw junction calling process, it's recommended to use the &quot;annotated&quot; output files instead which assign matching HUGO gene symbols based on the UCSC refGene table.</td>
    </tr>
    <tr class="odd">
    <td>transcripts</td>
    <td>list of known transcript IDs matching the junction.</td>
    </tr>
    <tr class="even">
    <td>qc_flanking</td>
    <td>Count of supporting reads passing flanking sequence checks (junctions observed adjacent to read ends require 18+ nt of flanking sequence, otherwise 10+ nt).</td>
    </tr>
    <tr class="odd">
    <td>qc_plus</td>
    <td>Count of supporting reads aligned to the + strand.</td>
    </tr>
    <tr class="even">
    <td>qc_minus</td>
    <td>Count of supporting reads aligned to the - strand.</td>
    </tr>
    <tr class="odd">
    <td>qc_perfect_reads</td>
    <td>Count of supporting reads with perfect alignments (no reference mismatches of quality 15+, indels, or soft clips).</td>
    </tr>
    <tr class="even">
    <td>qc_clean_reads</td>
    <td>Count of supporting reads whose alignments are not perfect but which have a ratio of &lt;= 5% of reference mismatches of quality 15+, indels, or soft clips relative to the count of aligned bases on both the left and right flanking sequence. Note: qc_clean_reads does NOT include qc_perfect_reads: to get a count of &quot;perfect plus pretty good&quot; reads the two values must be added together.</td>
    </tr>
    </tbody>
    </table>

### ProteinPaint Visualizations

-   **Interactive fusion visualization**: The interactive fusion viewer
    is available as a file with the name
    `Putative Fusions (SAMPLE.final_fusions)`. Click on this file and
    then follow the <span role="ref">pp-fusion-viewer</span> guide.
-   **Interactive coverage visualization**: The interactive coverage
    viewer is available as a file with the name `Coverage (SAMPLE)`.
    Click on this file and then follow the <span
    role="ref">pp-coverage-viewer</span> guide.

## Known Issues

### Adapter contamination

This pipeline does not, at present, remove adapter sequences. If the
sequencing library is contaminated with adapters, CICERO runtimes can
increase exponentially. We recommend running FastQ files through a QC
pipeline such as FastQC and trimming adapters with tools such as
Trimmomatic if adapters are found.

### High coverage regions

Certain cell types show very high transcription of certain loci, for
example, the immunoglobulin heavy chain locus in plasma cells. The
presence of very highly covered regions (typically 100,000-1,000,000+ X)
has an adverse effect on CICERO runtimes. Presently, we have no good
solution to this problem as strategies such as down-sampling may reduce
sensitivity over important regions of the genome.

### Interactive Visualizations Exon vs Intron Nomenclature

When a codon is split over a fusion gene junction, the annotation
software marks the event as intronic when really, the event should be
exonic. We are working to fix this bug. In the mean time, if a fusion is
predicted to be in frame but the interactive plot shows "intronic", we
suggest the user blat the contig shown just below to clarify if the true
junction is either in the intron or exon.

## Frequently Asked Questions

None yet! If you have any questions not covered here, feel free to reach
out on [our contact form](https://hospital.stjude.org/apps/forms/fb/st-jude-cloud-contact/).
