



<!DOCTYPE html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      <meta http-equiv="x-ua-compatible" content="ie=edge">
      
      
      
      
        <meta name="lang:clipboard.copy" content="Copy to clipboard">
      
        <meta name="lang:clipboard.copied" content="Copied to clipboard">
      
        <meta name="lang:search.language" content="en">
      
        <meta name="lang:search.pipeline.stopwords" content="True">
      
        <meta name="lang:search.pipeline.trimmer" content="True">
      
        <meta name="lang:search.result.none" content="No matching documents">
      
        <meta name="lang:search.result.one" content="1 matching document">
      
        <meta name="lang:search.result.other" content="# matching documents">
      
        <meta name="lang:search.tokenizer" content="[\s\-]+">
      
      <link rel="shortcut icon" href="../../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-0.17.4, mkdocs-material-2.9.2">
    
    
      
        <title>cis-X - St. Jude Cloud Documentation</title>
      
    
    
      <link rel="stylesheet" href="../../../assets/stylesheets/application.css">
      
      
    
    
      <script src="../../../assets/javascripts/modernizr.js"></script>
    
    
      <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
      
    
    <link rel="stylesheet" href="../../../assets/fonts/material-icons.css">
    
      <link rel="stylesheet" href="../../../css/extra.css">
    
    
  </head>
  
    <body dir="ltr">
  
    <svg class="md-svg">
      <defs>
        
        
          <svg xmlns="http://www.w3.org/2000/svg" width="416" height="448"
    viewBox="0 0 416 448" id="__github">
  <path fill="currentColor" d="M160 304q0 10-3.125 20.5t-10.75 19-18.125
        8.5-18.125-8.5-10.75-19-3.125-20.5 3.125-20.5 10.75-19 18.125-8.5
        18.125 8.5 10.75 19 3.125 20.5zM320 304q0 10-3.125 20.5t-10.75
        19-18.125 8.5-18.125-8.5-10.75-19-3.125-20.5 3.125-20.5 10.75-19
        18.125-8.5 18.125 8.5 10.75 19 3.125 20.5zM360
        304q0-30-17.25-51t-46.75-21q-10.25 0-48.75 5.25-17.75 2.75-39.25
        2.75t-39.25-2.75q-38-5.25-48.75-5.25-29.5 0-46.75 21t-17.25 51q0 22 8
        38.375t20.25 25.75 30.5 15 35 7.375 37.25 1.75h42q20.5 0
        37.25-1.75t35-7.375 30.5-15 20.25-25.75 8-38.375zM416 260q0 51.75-15.25
        82.75-9.5 19.25-26.375 33.25t-35.25 21.5-42.5 11.875-42.875 5.5-41.75
        1.125q-19.5 0-35.5-0.75t-36.875-3.125-38.125-7.5-34.25-12.875-30.25-20.25-21.5-28.75q-15.5-30.75-15.5-82.75
        0-59.25 34-99-6.75-20.5-6.75-42.5 0-29 12.75-54.5 27 0 47.5 9.875t47.25
        30.875q36.75-8.75 77.25-8.75 37 0 70 8 26.25-20.5
        46.75-30.25t47.25-9.75q12.75 25.5 12.75 54.5 0 21.75-6.75 42 34 40 34
        99.5z" />
</svg>
        
      </defs>
    </svg>
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" data-md-component="overlay" for="__drawer"></label>
    
      <a href="#overview" tabindex="1" class="md-skip">
        Skip to content
      </a>
    
    
      <header class="md-header" data-md-component="header">
  <nav class="md-header-nav md-grid">
    <div class="md-flex">
      <div class="md-flex__cell md-flex__cell--shrink">
        <a href="../../.." title="St. Jude Cloud Documentation" class="md-header-nav__button md-logo">
          
            <img src="../../../images/stjude-logo-child-white.png" width="40" height="28">
          
        </a>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        <label class="md-icon md-icon--menu md-header-nav__button" for="__drawer"></label>
      </div>
      <div class="md-flex__cell md-flex__cell--stretch">
        <div class="md-flex__ellipsis md-header-nav__title" data-md-component="title">
          
            
              <span class="md-header-nav__topic">
                St. Jude Cloud Documentation
              </span>
              <span class="md-header-nav__topic">
                cis-X
              </span>
            
          
        </div>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        
          
            <label class="md-icon md-icon--search md-header-nav__button" for="__search"></label>
            
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="query" data-md-state="active">
      <label class="md-icon md-search__icon" for="__search"></label>
      <button type="reset" class="md-icon md-search__icon" data-md-component="reset" tabindex="-1">
        &#xE5CD;
      </button>
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="result">
          <div class="md-search-result__meta">
            Type to start searching
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
          
        
      </div>
      
        <div class="md-flex__cell md-flex__cell--shrink">
          <div class="md-header-nav__source">
            


  


  <a href="https://github.com/stjude/sjcloud-docs/" title="Go to repository" class="md-source" data-md-source="github">
    
      <div class="md-source__icon">
        <svg viewBox="0 0 24 24" width="24" height="24">
          <use xlink:href="#__github" width="24" height="24"></use>
        </svg>
      </div>
    
    <div class="md-source__repository">
      stjude/sjcloud-docs
    </div>
  </a>

          </div>
        </div>
      
    </div>
  </nav>
</header>
    
    <div class="md-container">
      
        
      
      
      <main class="md-main">
        <div class="md-main__inner md-grid" data-md-component="container">
          
            
              <div class="md-sidebar md-sidebar--primary" data-md-component="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    <nav class="md-nav md-nav--primary" data-md-level="0">
  <label class="md-nav__title md-nav__title--site" for="__drawer">
    <span class="md-nav__button md-logo">
      
        <img src="../../../images/stjude-logo-child-white.png" width="48" height="48">
      
    </span>
    St. Jude Cloud Documentation
  </label>
  
    <div class="md-nav__source">
      


  


  <a href="https://github.com/stjude/sjcloud-docs/" title="Go to repository" class="md-source" data-md-source="github">
    
      <div class="md-source__icon">
        <svg viewBox="0 0 24 24" width="24" height="24">
          <use xlink:href="#__github" width="24" height="24"></use>
        </svg>
      </div>
    
    <div class="md-source__repository">
      stjude/sjcloud-docs
    </div>
  </a>

    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      


  <li class="md-nav__item">
    <a href="../../.." title="Getting Started" class="md-nav__link">
      Getting Started
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-2" type="checkbox" id="nav-2">
    
    <label class="md-nav__link" for="nav-2">
      Data Guides
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-2">
        Data Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../../data/data-overview/" title="Overview" class="md-nav__link">
      Overview
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../data/types-of-data/" title="Types of Data" class="md-nav__link">
      Types of Data
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../data/data-request/" title="Making a Data Request" class="md-nav__link">
      Making a Data Request
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../data/data-access-units/" title="Data Access Units" class="md-nav__link">
      Data Access Units
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../data/metadata/" title="Metadata Provided" class="md-nav__link">
      Metadata Provided
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../data/run-your-tools/" title="Working with our Data" class="md-nav__link">
      Working with our Data
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../data/data-transfer-app/" title="Data Transfer Application" class="md-nav__link">
      Data Transfer Application
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../data/command-line/" title="Command Line Interaction" class="md-nav__link">
      Command Line Interaction
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      

  


  <li class="md-nav__item md-nav__item--active md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-3" type="checkbox" id="nav-3" checked>
    
    <label class="md-nav__link" for="nav-3">
      Tool Guides
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-3">
        Tool Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../pecan-pie/" title="Pecan PIE" class="md-nav__link">
      Pecan PIE
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../neoepitope/" title="NeoepitopePred" class="md-nav__link">
      NeoepitopePred
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../chipseq/" title="ChIP-Seq Peak Calling" class="md-nav__link">
      ChIP-Seq Peak Calling
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../rapid-rnaseq/" title="Rapid RNA-Seq Fusion Detection" class="md-nav__link">
      Rapid RNA-Seq Fusion Detection
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../warden/" title="WARDEN Differential Expression Analysis" class="md-nav__link">
      WARDEN Differential Expression Analysis
    </a>
  </li>

        
          
          
          

  


  <li class="md-nav__item md-nav__item--active">
    
    <input class="md-toggle md-nav__toggle" data-md-toggle="toc" type="checkbox" id="__toc">
    
    
      <label class="md-nav__link md-nav__link--active" for="__toc">
        cis-X
      </label>
    
    <a href="./" title="cis-X" class="md-nav__link md-nav__link--active">
      cis-X
    </a>
    
      
<nav class="md-nav md-nav--secondary">
  
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#overview" title="Overview" class="md-nav__link">
    Overview
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#getting-started" title="Getting started" class="md-nav__link">
    Getting started
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#input-file-configuration" title="Input file configuration" class="md-nav__link">
    Input file configuration
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#uploading-data" title="Uploading data" class="md-nav__link">
    Uploading data
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#running-the-tool" title="Running the tool" class="md-nav__link">
    Running the tool
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#monitoring-run-progress" title="Monitoring run progress" class="md-nav__link">
    Monitoring run progress
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#analysis-of-results" title="Analysis of results" class="md-nav__link">
    Analysis of results
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#interpreting-results" title="Interpreting results" class="md-nav__link">
    Interpreting results
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
    
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../mutational-spectrum/" title="Mutational Spectrum" class="md-nav__link">
      Mutational Spectrum
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-4" type="checkbox" id="nav-4">
    
    <label class="md-nav__link" for="nav-4">
      Visualization Guides
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-4">
        Visualization Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../../visualizations/bigwig-viewer/" title="BigWig Viewer" class="md-nav__link">
      BigWig Viewer
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../../visualizations/fusion-viewer/" title="Fusion Viewer" class="md-nav__link">
      Fusion Viewer
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-toggle md-nav__toggle" data-md-toggle="nav-5" type="checkbox" id="nav-5">
    
    <label class="md-nav__link" for="nav-5">
      Portal Guides
    </label>
    <nav class="md-nav" data-md-component="collapsible" data-md-level="1">
      <label class="md-nav__title" for="nav-5">
        Portal Guides
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../../portals/sickle-cell/" title="Sickle Cell Genomics Portal" class="md-nav__link">
      Sickle Cell Genomics Portal
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../../../faq/" title="Frequently Asked Questions" class="md-nav__link">
      Frequently Asked Questions
    </a>
  </li>

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              <div class="md-sidebar md-sidebar--secondary" data-md-component="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary">
  
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#overview" title="Overview" class="md-nav__link">
    Overview
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#getting-started" title="Getting started" class="md-nav__link">
    Getting started
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#input-file-configuration" title="Input file configuration" class="md-nav__link">
    Input file configuration
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#uploading-data" title="Uploading data" class="md-nav__link">
    Uploading data
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#running-the-tool" title="Running the tool" class="md-nav__link">
    Running the tool
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#monitoring-run-progress" title="Monitoring run progress" class="md-nav__link">
    Monitoring run progress
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#analysis-of-results" title="Analysis of results" class="md-nav__link">
    Analysis of results
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#interpreting-results" title="Interpreting results" class="md-nav__link">
    Interpreting results
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content">
            <article class="md-content__inner md-typeset">
              
                
                  <a href="https://github.com/stjude/sjcloud-docs/blob/cmcleod/overhaul/docs/guides/tools/cis-x.md" title="Edit this page" class="md-icon md-content__icon">&#xE3C9;</a>
                
                
                  <h1>cis-X</h1>
                
                <div class="admonition warning">
<p class="admonition-title">Warning</p>
<p>cis-X is an upcoming St. Jude Cloud tool and is not yet publicly available.
See <a href="https://www.stjuderesearch.org/site/lab/zhang/cis-x">cis-X on St. Jude Research</a> for more information.</p>
</div>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Authors</strong></td>
<td>Yu Liu, Chunliang Li, Shuhong Shen</td>
</tr>
<tr>
<td><strong>Publication</strong></td>
<td>N/A (not published)</td>
</tr>
<tr>
<td><strong>Technical Support</strong></td>
<td><a href="https://stjude.cloud/contact">Contact Us</a></td>
</tr>
</tbody>
</table>
<p>Activating regular variants usually cause the cis-activation of target genes.
To find cis-activated genes, allelic specific/imbalance expressions (ASE) and
outlier high expression (OHE) signals are used. Variants in the same
topologically associated domains with the candidates can then be searched,
including structural variants (SV), copy number aberrations (CNA), and single
nucleotide variations (SNV) and insertion/deletions (indel).</p>
<p>A transcription factor binding analysis is also done, using motifs from
<a href="http://hocomoco11.autosome.ru/">HOCOMOCO</a> v10 models.</p>
<p>cis-X currently only works with hg19 (GRCh37).</p>
<h2 id="overview">Overview</h2>
<h3 id="inputs">Inputs</h3>

<table>
<thead>
<tr>
<th>Name</th>
<th>Type</th>
<th>Description</th>
<th>Example</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sample ID</td>
<td>String</td>
<td>The ID of the input sample</td>
<td>SJALL018373_D1</td>
</tr>
<tr>
<td>Disease subtype</td>
<td>String</td>
<td>The disease name under analysis. Must be either TALL or AML.</td>
<td>TALL</td>
</tr>
<tr>
<td><a href="#single-nucleotide-variants">Single nucleotide variants</a></td>
<td>File</td>
<td>Tab-delimited file containing raw sequence variants</td>
<td>*.txt</td>
</tr>
<tr>
<td><a href="#cnvloh-regions">CNV/LOH regions</a></td>
<td>File</td>
<td>Tab-delimited file containing any aneuploidy region existing in the tumor genome under analysis</td>
<td>*.txt</td>
</tr>
<tr>
<td>RNA-seq BAM</td>
<td>File</td>
<td>BAM file aligned to hg19 (GRCh37)</td>
<td>*.bam</td>
</tr>
<tr>
<td>RNA-seq BAM index</td>
<td>File</td>
<td>BAM index for the given BAM</td>
<td>*.bam.bai</td>
</tr>
<tr>
<td><a href="#gene-expression-table">Gene expression table</a></td>
<td>File</td>
<td>Tab-delimited file containing gene level expressions for the tumor under analysis in FPKM</td>
<td>*.txt</td>
</tr>
<tr>
<td><a href="#somatic-snvindels">Somatic SNV/indels</a></td>
<td>File</td>
<td>Tab-delimited file containing somatic SNV/indels in the tumor genome</td>
<td>*.txt</td>
</tr>
<tr>
<td><a href="#somatic-svs">Somatic SVs</a></td>
<td>File</td>
<td>Tab-delimited file containing somatic acquired structural variants in the tumor genome</td>
<td>*.txt</td>
</tr>
<tr>
<td><a href="#somatic-cnvs">Somatic CNVs</a></td>
<td>File</td>
<td>Tab-delimited file containing copy number aberrations in the tumor genome</td>
<td>*.txt</td>
</tr>
<tr>
<td>CNV/LOH action</td>
<td>String</td>
<td>The behavior when handling markers in CNV/LOH regions. Can be either <code class="codehilite">keep</code> or <code class="codehilite">drop</code>.</td>
<td>drop</td>
</tr>
<tr>
<td>Minimum coverage for WGS</td>
<td>Integer</td>
<td>The minimum coverage in WGS to be included in the analysis</td>
<td>10</td>
</tr>
<tr>
<td>Minimum coverage for RNA-seq</td>
<td>Integer</td>
<td>The minimum coverage in RNA-seq to be included in the analysis</td>
<td>5</td>
</tr>
<tr>
<td>Candidate FPKM threshold</td>
<td>Float</td>
<td>The FPKM threshold for the nomination of a cis-activated candidate</td>
<td>0.1</td>
</tr>
</tbody>
</table>
<h3 id="outputs">Outputs</h3>

<table>
<thead>
<tr>
<th>Name</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="#cis-activated-candidates">cis-activated candidates</a></td>
<td>cis-activated candidates in the tumor genome under analysis</td>
</tr>
<tr>
<td><a href="#sv-candidates">SV candidates</a></td>
<td>Structural variant (SV) candidates predicted as the causal for the cis-activated genes in the regulatory territory</td>
</tr>
<tr>
<td><a href="#cna-candidates">CNA candidates</a></td>
<td>Copy number aberrations (CNA) predicted as the causal for the cis-activated genes in the regulatory territory</td>
</tr>
<tr>
<td><a href="#snvindel-candidates">SNV/indel candidates</a></td>
<td>SNV/indel candidates predicted as functional and predicted transcription factors</td>
</tr>
<tr>
<td><a href="#ohe-results">OHE results</a></td>
<td>Raw outlier high expression (OHE) results</td>
</tr>
<tr>
<td><a href="#gene-level-ase-results">Gene level ASE results</a></td>
<td>Raw gene level allelic specific expression (ASE) results</td>
</tr>
<tr>
<td><a href="#single-marker-ase-results">Single marker ASE results</a></td>
<td>Raw single marker allelic specific expression (ASE) results</td>
</tr>
</tbody>
</table>
<h2 id="getting-started">Getting started</h2>
<div class="admonition todo">
<p class="admonition-title">Todo</p>
</div>
<h3 id="input-file-configuration">Input file configuration</h3>
<p>cis-X requires six tab-delimited input files to be prepared in advance.</p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>Even though CNV/LOH regions, somatic SNV/indels, somatic SVs, and
somatic CNVs can be "empty", using such inputs will produce results with a
much higher false positive rate.</p>
</div>
<h4 id="single-nucleotide-variants">Single nucleotide variants</h4>

<p>A list of single nucleotide markers is a tab-delimited file with the
following columns:</p>
<ul>
<li><code class="codehilite">Chr</code>: chromosome name for the marker</li>
<li><code class="codehilite">Pos</code>: genomic start location for the marker</li>
<li><code class="codehilite">Chr_Allele</code>: reference allele</li>
<li><code class="codehilite">Alternative_Allele</code>: alternative allele</li>
<li><code class="codehilite">reference_tumor_count</code>: reference allele count in the tumor genome</li>
<li><code class="codehilite">alternative_tumor_count</code>: alterative allele count in the tumor genome</li>
<li><code class="codehilite">reference_normal_count</code>: reference allele count in the matched normal genome</li>
<li><code class="codehilite">alternative_normal_count</code>: alternative count in the matched normal genome</li>
</ul>
<p>This file can be generated with Bambino.</p>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>Chr</th>
<th align="right">Pos</th>
<th>Chr_Allele</th>
<th>Alternative_Allele</th>
<th align="right">reference_tumor_count</th>
<th align="right">alternative_tumor_count</th>
<th align="right">reference_normal_count</th>
<th>alternative_normal_count</th>
</tr>
</thead>
<tbody>
<tr>
<td>chr11</td>
<td align="right">61396</td>
<td>TT</td>
<td></td>
<td align="right">0</td>
<td align="right">3</td>
<td align="right">0</td>
<td>10</td>
</tr>
<tr>
<td>chr11</td>
<td align="right">72981</td>
<td></td>
<td>T</td>
<td align="right">1</td>
<td align="right">3</td>
<td align="right">2</td>
<td>3</td>
</tr>
</tbody>
</table>
<h4 id="cnvloh-regions">CNV/LOH regions</h4>

<p>The CNV/LOH regions are all the genomic regions carrying copy number
variations (CNV) or loss of heterozygosity (LOH), which will be filtered out
during analysis.</p>
<p>This is a tab-delimited file in the bed format. It must have at least the
following three columns:</p>
<ul>
<li><code class="codehilite">chrom</code>: chromosome name</li>
<li><code class="codehilite">loc.start</code>: genomic start location</li>
<li><code class="codehilite">loc.end</code>: genomic end location</li>
</ul>
<p>If no CNV/LOH are in the genome under analysis, a file with no rows (but
including headers) can be provided.</p>
<p>This file can be generated with CONSERTING.</p>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>chrom</th>
<th align="right">loc.start</th>
<th align="right">loc.end</th>
<th>Sample</th>
<th align="right">seg.mean</th>
<th align="right">LogRatio</th>
<th>source</th>
</tr>
</thead>
<tbody>
<tr>
<td>chr9</td>
<td align="right">10712</td>
<td align="right">37855747</td>
<td>SJALL018373_D1</td>
<td align="right">0.471181417</td>
<td align="right"></td>
<td>LOH</td>
</tr>
<tr>
<td>chr9</td>
<td align="right">20276901</td>
<td align="right">20703900</td>
<td>SJALL018373_D1</td>
<td align="right">-0.978</td>
<td align="right">-5.696</td>
<td>CNV</td>
</tr>
</tbody>
</table>
<h4 id="gene-expression-table">Gene expression table</h4>

<p>The gene expression table is a tab-delimited file containing gene level
expressions for the tumor under analysis. The expressions are in FPKM
(fragments per kilobase of transcript per million mapped reads).</p>
<ul>
<li><code class="codehilite">GeneID</code>: gene <a href="http://www.ensembl.org/">Ensembl</a> ID</li>
<li><code class="codehilite">GeneName</code>: gene symbol</li>
<li><code class="codehilite">Type</code>: <a href="https://www.gencodegenes.org/gencode_biotypes.html">transcript type</a></li>
<li><code class="codehilite">Status</code>: transcript status (must be <code class="codehilite">KNOWN</code>, <code class="codehilite">NOVEL</code>, or <code class="codehilite">PUTATIVE</code>)</li>
<li><code class="codehilite">Chr</code>: chromosome name</li>
<li><code class="codehilite">Start</code> genomic start location</li>
<li><code class="codehilite">End</code>: genomic end location</li>
<li>[SampleID...]: FPKM for the given sample</li>
</ul>
<p>This file can be generated with the output of HTseq-count preprocessed
through <code class="codehilite">mergeData_geneName.pl</code> (available with the distribution of cis-X).
The data must be able to match values in the given gene specific reference
expression matrices generated from a larger cohort.</p>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>GeneID</th>
<th>GeneName</th>
<th>Type</th>
<th>Status</th>
<th>Chr</th>
<th align="right">Start</th>
<th align="right">End</th>
<th align="right">SJALL018373_D1</th>
</tr>
</thead>
<tbody>
<tr>
<td>ENSG00000261122.2</td>
<td>5S_rRNA</td>
<td>lincRNA</td>
<td>NOVEL</td>
<td>chr16</td>
<td align="right">34977639</td>
<td align="right">34990886</td>
<td align="right">0.0000</td>
</tr>
<tr>
<td>ENSG00000249352.3</td>
<td>7SK</td>
<td>lincRNA</td>
<td>NOVEL</td>
<td>chr5</td>
<td align="right">68266266</td>
<td align="right">68325992</td>
<td align="right">4.5937</td>
</tr>
</tbody>
</table>
<h4 id="somatic-snvindels">Somatic SNV/indels</h4>

<p>This is a tab-delimited file containing somatic sequence mutations present in
the genome under analysis. It includes both single nucleotide variants (SNV)
and small insertion/deletions (indel). The file must have the following
columns:</p>
<ul>
<li><code class="codehilite">chr</code>: chromosome name</li>
<li><code class="codehilite">pos</code>: genomic start location</li>
<li><code class="codehilite">ref</code>: reference nucleotide</li>
<li><code class="codehilite">mutant</code>: mutant nucleotide</li>
<li><code class="codehilite">type</code>: mutation type (must be either <code class="codehilite">snv</code> or <code class="codehilite">indel</code>)</li>
</ul>
<p>Note that the coordinate used for an indel is after the inserted sequence.</p>
<p>If no SNV/indels are in the sample under analysis, a file with no rows
(but including headers) can be provided.</p>
<p>This file can can be created with Bambino and then preprocessed using the
steps taken in "<a href="https://www.ncbi.nlm.nih.gov/pubmed/22237106">The genetic basis of early T-cell precursor acute lymphoblastic leukaemia</a>".</p>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>chr</th>
<th align="right">pos</th>
<th>ref</th>
<th>mut</th>
<th>type</th>
</tr>
</thead>
<tbody>
<tr>
<td>chr1</td>
<td align="right">24782720</td>
<td>G</td>
<td>A</td>
<td>snv</td>
</tr>
<tr>
<td>chr11</td>
<td align="right">82896176</td>
<td>T</td>
<td>C</td>
<td>snv</td>
</tr>
</tbody>
</table>
<h4 id="somatic-svs">Somatic SVs</h4>

<p>This is a tab-delimited file containing somatic-acquired structural variants
(SV) in the cancer genome. The file must have the following columns:</p>
<ul>
<li><code class="codehilite">chrA</code>: chromosome name of the left breakpoint</li>
<li><code class="codehilite">posA</code>: genomic location of the left breakpoint</li>
<li><code class="codehilite">ortA</code>: strand orientation of the left breakpoint</li>
<li><code class="codehilite">chrB</code>: chromosome name of the right breakpoint</li>
<li><code class="codehilite">posB</code>: genomic location of the right breakpoint</li>
<li><code class="codehilite">ortB</code>: strand orientation of the right breakpoint</li>
</ul>
<p>Strand orientations are denoted with a <code class="codehilite">+</code> for a sense or coding strand and
<code class="codehilite">-</code> for a antisense or non-coding strand.</p>
<p>If no somatic SVs are in the sample under analysis, a file with no rows (but
including headers) can be provided.</p>
<p>This file can be generated by CREST.</p>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>chrA</th>
<th align="right">posA</th>
<th>ortA</th>
<th>chrB</th>
<th align="right">posB</th>
<th>ortB</th>
<th>type</th>
</tr>
</thead>
<tbody>
<tr>
<td>chr11</td>
<td align="right">33913169</td>
<td>+</td>
<td>chr7</td>
<td align="right">142494049</td>
<td>-</td>
<td>CTX</td>
</tr>
<tr>
<td>chr11</td>
<td align="right">64219334</td>
<td>+</td>
<td>chr2</td>
<td align="right">205042527</td>
<td>-</td>
<td>CTX</td>
</tr>
</tbody>
</table>
<h4 id="somatic-cnvs">Somatic CNVs</h4>

<p>This is a tab-delimited file containing the genomic
regions with somatic-acquired copy number aberrations (CNA) in the cancer
genome.</p>
<ul>
<li><code class="codehilite">chr</code>: chromosome name</li>
<li><code class="codehilite">start</code>: genomic start location</li>
<li><code class="codehilite">end</code>: genomic end location</li>
<li><code class="codehilite">logR</code>: log2 ratio</li>
</ul>
<p>If no somatic CNVs are in the sample under analysis, a file with no rows
(but including headers) can be provided.</p>
<p>This file can be generating by CONSERTING.</p>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>chr</th>
<th align="right">start</th>
<th align="right">end</th>
<th align="right">logR</th>
</tr>
</thead>
<tbody>
<tr>
<td>chr9</td>
<td align="right">20276901</td>
<td align="right">20703900</td>
<td align="right">-5.696</td>
</tr>
</tbody>
</table>
<h2 id="uploading-data">Uploading data</h2>
<p>cis-X requires a total of eight files to be uploaded, as described in "<a href="#input-file-configuration">Input
file configuration</a>". These files can be uploaded via the <a href="../../data/data-transfer-app/">data transfer
application</a> or <a href="../../data/command-line/">command line</a>.</p>
<h2 id="running-the-tool">Running the tool</h2>
<div class="admonition todo">
<p class="admonition-title">Todo</p>
</div>
<h2 id="monitoring-run-progress">Monitoring run progress</h2>
<div class="admonition todo">
<p class="admonition-title">Todo</p>
</div>
<h2 id="analysis-of-results">Analysis of results</h2>
<p>Upon a successful run of cis-X, seven tab-delimited files are saved to the
results directory. These raw results can be processed through external tools
for further analysis.</p>
<h3 id="interpreting-results">Interpreting results</h3>
<h4 id="cis-activated-candidates">cis-activated candidates</h4>

<p>The main result file contains the cis-activated candidates in the tumor genome
under analysis.</p>
<ul>
<li><code class="codehilite">gene</code>: gene accession number (<a href="https://www.ncbi.nlm.nih.gov/refseq/">RefSeq</a> ID)</li>
<li><code class="codehilite">gsym</code>: gene symbol</li>
<li><code class="codehilite">chrom</code>: chromosome name</li>
<li><code class="codehilite">strand</code>: strand orientation</li>
<li><code class="codehilite">start</code>: genomic start location</li>
<li><code class="codehilite">end</code>: genomic end location</li>
<li><code class="codehilite">cdsStartStat</code>: coding sequence (CDS) start status</li>
<li><code class="codehilite">cdsEndStat</code>: coding sequence (CDS) end status</li>
<li><code class="codehilite">markers</code>: number of heterozygous markers in this gene</li>
<li><code class="codehilite">ase_markers</code>: number of heterozygous markers showing allelic specific expressions (ASE)</li>
<li><code class="codehilite">average_ai_all</code>: average B-allele frequency (BAF) difference between RNA and DNA for all heterozygous markers</li>
<li><code class="codehilite">average_ai_ase</code>: average BAF difference between RNA and DNA for ASE markers</li>
<li><code class="codehilite">pval_all_markers</code>: p-value for each marker in the ASE test</li>
<li><code class="codehilite">pval_ase_markers</code>: p-value for ASE markers in the ASE test</li>
<li><code class="codehilite">ai_all_markers</code>: BAF difference between RNA and DNA for all heterozygrous markers</li>
<li><code class="codehilite">ai_ase_markers</code>: BAF difference between RNA and DNA for ASE markers</li>
<li><code class="codehilite">comb.pval</code>: combined p-value for the ASE test</li>
<li><code class="codehilite">mean.delta</code>: average BAF difference between RNA and DNA for all markers</li>
<li><code class="codehilite">rawp</code>: raw p-value for the ASE test</li>
<li><code class="codehilite">Bonferroni</code>: adjusted p-value for the ASE test (single-step Bonferroni)</li>
<li><code class="codehilite">ABH</code>: adjusted p-value for the ASE test (Benjamini-Hochberg)</li>
<li><code class="codehilite">FPKM</code>: FPKM value</li>
<li><code class="codehilite">loo.source</code>: which reference expression matrix was used in the outlier high expression (OHE) test</li>
<li><code class="codehilite">loo.cohort.size</code>: number of cases in the reference expression matrix for this gene</li>
<li><code class="codehilite">loo.pval</code>: p-value of the OHE test</li>
<li><code class="codehilite">loo.rank</code>: rank of the case under analysis among the reference cases</li>
<li><code class="codehilite">imprinting.status</code>: imprinting status of the gene</li>
<li><code class="codehilite">candidate.group</code>: status of the gene, combining both ASE and outlier tests</li>
</ul>
<p>Strand orientations are denoted with a <code class="codehilite">+</code> for a sense or coding strand and
<code class="codehilite">-</code> for a antisense or non-coding strand.</p>
<p>Coding sequence status is typically one of "none" (not specified), "unk"
(unknown), "incmpl" (incomplete), or "cmpl" (complete).</p>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>gene</th>
<th>gsym</th>
<th>chrom</th>
<th>strand</th>
<th align="right">start</th>
<th align="right">end</th>
<th>cdsStartStat</th>
<th>cdsEndStat</th>
<th align="right">markers</th>
<th align="right">ase_markers</th>
<th align="right">average_ai_all</th>
<th align="right">average_ai_ase</th>
<th>pval_all_markers</th>
<th>pval_ase_markers</th>
<th>ai_all_markers</th>
<th>ai_ase_markers</th>
<th align="right">comb.pval</th>
<th align="right">mean.delta</th>
<th align="right">rawp</th>
<th align="right">Bonferroni</th>
<th align="right">ABH</th>
<th align="right">FPKM</th>
<th>loo.source</th>
<th align="right">loo.cohort.size</th>
<th align="right">loo.pval</th>
<th align="right">loo.rank</th>
<th>imprinting.status</th>
<th>candidate.group</th>
</tr>
</thead>
<tbody>
<tr>
<td>NM_145804</td>
<td>ABTB2</td>
<td>chr11</td>
<td>-</td>
<td align="right">34172533</td>
<td align="right">34379555</td>
<td>cmpl</td>
<td>cmpl</td>
<td align="right">5</td>
<td align="right">5</td>
<td align="right">0.5</td>
<td align="right">0.500</td>
<td>0.001953125,0.001953125,0.001953125,6.10351562500001e-05,0.000244140625</td>
<td>0.001953125,0.001953125,0.001953125,6.10351562500001e-05,0.000244140625</td>
<td>0.5,0.5,0.5,0.5,0.5</td>
<td>0.5,0.5,0.5,0.5,0.5</td>
<td align="right">0.000644290972057077</td>
<td align="right">0.5</td>
<td align="right">0.000644290972057077</td>
<td align="right">0.632049443587993</td>
<td align="right">0.0110866672927557</td>
<td align="right">7.6776</td>
<td>bi_cohort</td>
<td align="right">40</td>
<td align="right">0.0367241086505276</td>
<td align="right">1</td>
<td></td>
<td>ase_outlier</td>
</tr>
<tr>
<td>NM_003189</td>
<td>TAL1</td>
<td>chr1</td>
<td>-</td>
<td align="right">47681961</td>
<td align="right">47698007</td>
<td>cmpl</td>
<td>cmpl</td>
<td align="right">2</td>
<td align="right">2</td>
<td align="right">0.482</td>
<td align="right">0.482</td>
<td>6.66361745922277e-28,3.30872245021211e-24</td>
<td>6.66361745922277e-28,3.30872245021211e-24</td>
<td>0.464912280701754,0.5</td>
<td>0.464912280701754,0.5</td>
<td align="right">4.69553625126628e-26</td>
<td align="right">0.482456140350877</td>
<td align="right">4.69553625126628e-26</td>
<td align="right">4.60632106249222e-23</td>
<td align="right">6.11761294450693e-24</td>
<td align="right">8.8168</td>
<td>white_list</td>
<td align="right">167</td>
<td align="right">0.0139385771987089</td>
<td align="right">1</td>
<td></td>
<td>ase_outlier</td>
</tr>
</tbody>
</table>
<h4 id="sv-candidates">SV candidates</h4>

<p>Structural variant (SV) candidates include candidates predicted as the causal
for the cis-activated genes in the regulatory territory.</p>
<ul>
<li><code class="codehilite">left.candidate.inTAD</code>: cis-activated candidate near the left breakpoint</li>
<li><code class="codehilite">right.candidate.inTAD</code>: cis-activated candidate near the right breakpoint</li>
<li><code class="codehilite">chrA</code>: chromosome name of the left breakpoint</li>
<li><code class="codehilite">posA</code>: genomic location of the left breakpoint</li>
<li><code class="codehilite">ortA</code>: strand orientation of the left breakpoint</li>
<li><code class="codehilite">chrB</code>: chromosome name of the right breakpoint</li>
<li><code class="codehilite">posB</code>: genomic location of the right breakpoint</li>
<li><code class="codehilite">ortB</code>: strand orientation of the right breakpoint</li>
<li><code class="codehilite">type</code>: type of translocation</li>
</ul>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>left.candidate.inTAD</th>
<th>right.candidate.inTAD</th>
<th>chrA</th>
<th align="right">posA</th>
<th>ortA</th>
<th>chrB</th>
<th align="right">posB</th>
<th>ortB</th>
<th>type</th>
</tr>
</thead>
<tbody>
<tr>
<td>LMO2</td>
<td></td>
<td>chr11</td>
<td align="right">33913169</td>
<td>+</td>
<td>chr7</td>
<td align="right">142494049</td>
<td>-</td>
<td>CTX</td>
</tr>
</tbody>
</table>
<h4 id="cna-candidates">CNA candidates</h4>

<p>Copy number aberration (CNA) candidates include candidates predicted as the
causal for the cis-activated genes in the regulatory territory.</p>
<ul>
<li><code class="codehilite">candidate.inTAD</code>: cis-activated candidate by the CNA</li>
<li><code class="codehilite">chr</code>: chromosome name</li>
<li><code class="codehilite">start</code>: genomic start position</li>
<li><code class="codehilite">end</code>: genomic end location</li>
<li><code class="codehilite">logR</code>: log ratio of the CNA</li>
</ul>
<h4 id="snvindel-candidates">SNV/indel candidates</h4>

<p>SNV/indel candidates include predicted candidates as functional and predicted
transcription factors. The mutations are also annotated for known regulatory
elements reported by the <a href="https://egg2.wustl.edu/roadmap/web_portal/index.html">NIH Roadmap Epigenomics Project</a> by collecting 111
cell lines.</p>
<ul>
<li><code class="codehilite">chrom</code>: chromosome name</li>
<li><code class="codehilite">pos</code>: genomic start position</li>
<li><code class="codehilite">ref</code>: reference allele genotype</li>
<li><code class="codehilite">mut</code>: mutant allele genotype</li>
<li><code class="codehilite">type</code>: mutation type (either <code class="codehilite">snv</code> or <code class="codehilite">indel</code>)</li>
<li><code class="codehilite">target</code>: cis-activated candidate</li>
<li><code class="codehilite">dist</code>: distance between the mutation and transcription start sites of the target gene</li>
<li><code class="codehilite">tf</code>: transcription factors predicted to have the binding motif introduced by the mutation</li>
<li><code class="codehilite">EpiRoadmap_enhancer</code>: enhancer regions that overlap with the mutation (from the <a href="https://egg2.wustl.edu/roadmap/web_portal/index.html">NIH Roadmap Epigenomics Project</a>)</li>
<li><code class="codehilite">EpiRoadmap_promoter</code>: promoter regions that overlap with the mutation (from the <a href="https://egg2.wustl.edu/roadmap/web_portal/index.html">NIH Roadmap Epigenomics Project</a>)</li>
<li><code class="codehilite">EpiRoadmap_dyadic</code>: dyadic regions that overlap with the mutation (from the <a href="https://egg2.wustl.edu/roadmap/web_portal/index.html">NIH Roadmap Epigenomics Project</a>)</li>
</ul>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>chrom</th>
<th align="right">pos</th>
<th>ref</th>
<th>mut</th>
<th>type</th>
<th>target</th>
<th>dist</th>
<th>tf</th>
<th>EpiRoadmap_enhancer</th>
<th>EpiRoadmap_promoter</th>
<th>EpiRoadmap_dyadic</th>
</tr>
</thead>
<tbody>
<tr>
<td>chr1</td>
<td align="right">47696311</td>
<td>C</td>
<td>T</td>
<td>snv</td>
<td>TAL1</td>
<td>1696</td>
<td>BCL11A,CEBPG,PBX2,YY1,ZBTB4</td>
<td></td>
<td>Brain,Digestive,ES-deriv,ESC,HSC &amp; B-cell,Heart,Muscle,Other,Sm. Muscle,iPSC</td>
<td></td>
</tr>
</tbody>
</table>
<h4 id="ohe-results">OHE results</h4>

<p>OHE results are the raw results for the outlier expression test.</p>
<ul>
<li><code class="codehilite">Gene</code>: gene symbol</li>
<li><code class="codehilite">fpkm.raw</code>: FPKM value</li>
<li><code class="codehilite">size.bi</code>: number of cases in the bi-allelic reference cohort</li>
<li><code class="codehilite">p.bi</code>: p-value in the outlier test using the bi-allelic reference cohort</li>
<li><code class="codehilite">rank.bi</code>: rank of the expression level in the case under analysis compared to the bi-allelic reference cohort</li>
<li><code class="codehilite">size.cohort</code>: number of cases in the entire reference cohort</li>
<li><code class="codehilite">p.cohort</code>: p-value in the outlier test using the entire reference cohort</li>
<li><code class="codehilite">rank.cohort</code>: rank of the expression level in the case under analysis compared to the entire reference cohort</li>
<li><code class="codehilite">size.white</code>: number of cases in the whitelist reference cohort</li>
<li><code class="codehilite">p.white</code>: p-value in the outlier test using the whitelist reference cohort</li>
<li><code class="codehilite">rank.white</code>: rank of the expression level in the case under analysis compared to the whitelist reference cohort</li>
</ul>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>Gene</th>
<th align="right">fpkm.raw</th>
<th align="right">size.bi</th>
<th align="right">p.bi</th>
<th align="right">rank.bi</th>
<th align="right">size.cohort</th>
<th align="right">p.cohort</th>
<th align="right">rank.cohort</th>
<th align="right">size.white</th>
<th align="right">p.white</th>
<th align="right">rank.white</th>
</tr>
</thead>
<tbody>
<tr>
<td>7SK</td>
<td align="right">4.5937</td>
<td align="right">na</td>
<td align="right">na</td>
<td align="right">na</td>
<td align="right">264</td>
<td align="right">0.716284011918374</td>
<td align="right">162</td>
<td align="right">na</td>
<td align="right">na</td>
<td align="right">na</td>
</tr>
<tr>
<td>A1BG</td>
<td align="right">0.2312</td>
<td align="right">24</td>
<td align="right">0.900132642257996</td>
<td align="right">21</td>
<td align="right">264</td>
<td align="right">0.84055666600945</td>
<td align="right">222</td>
<td align="right">na</td>
<td align="right">na</td>
<td align="right">na</td>
</tr>
</tbody>
</table>
<h4 id="gene-level-ase-results">Gene level ASE results</h4>

<p>Gene level ASE results are the raw results from the gene level ASE test.</p>
<ul>
<li><code class="codehilite">gene</code>: gene accession number (<a href="https://www.ncbi.nlm.nih.gov/refseq/">RefSeq</a> ID)</li>
<li><code class="codehilite">gsym</code>: gene symbol</li>
<li><code class="codehilite">chrom</code>: chromosome name</li>
<li><code class="codehilite">strand</code>: strand orientation</li>
<li><code class="codehilite">start</code>: genomic start location</li>
<li><code class="codehilite">end</code>: genomic end location</li>
<li><code class="codehilite">cdsStartStat</code>: coding sequence (CDS) start status</li>
<li><code class="codehilite">cdsEndStat</code>: coding sequence (CDS) end status</li>
<li><code class="codehilite">markers</code>: number of heterozygous markers in this gene</li>
<li><code class="codehilite">ase_markers</code>: number of heterozygous markers showing allelic specific expressions (ASE)</li>
<li><code class="codehilite">average_ai_all</code>: average B-allele frequency (BAF) difference between RNA and DNA for all heterozygous markers</li>
<li><code class="codehilite">average_ai_ase</code>: average BAF difference between RNA and DNA for ASE markers</li>
<li><code class="codehilite">pval_all_markers</code>: p-value for each marker in the ASE test</li>
<li><code class="codehilite">pval_ase_markers</code>: p-value for ASE markers in the ASE test</li>
<li><code class="codehilite">ai_all_markers</code>: BAF difference between RNA and DNA for all heterozygrous markers</li>
<li><code class="codehilite">ai_ase_markers</code>: BAF difference between RNA and DNA for ASE markers</li>
<li><code class="codehilite">comb.pval</code>: combined p-value for the ASE test</li>
<li><code class="codehilite">mean.delta</code>: average BAF difference between RNA and DNA for all markers</li>
<li><code class="codehilite">rawp</code>: raw p-value for the ASE test</li>
<li><code class="codehilite">Bonferroni</code>: adjusted p-value for the ASE test (single-step Bonferroni)</li>
<li><code class="codehilite">ABH</code>: adjusted p-value for the ASE test (Benjamini-Hochberg)</li>
</ul>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>gene</th>
<th>gsym</th>
<th>chrom</th>
<th>strand</th>
<th align="right">start</th>
<th align="right">end</th>
<th>cdsStartStat</th>
<th>cdsEndStat</th>
<th align="right">markers</th>
<th align="right">ase_markers</th>
<th align="right">average_ai_all</th>
<th align="right">average_ai_ase</th>
<th>pval_all_markers</th>
<th>pval_ase_markers</th>
<th>ai_all_markers</th>
<th>ai_ase_markers</th>
<th align="right">comb.pval</th>
<th align="right">mean.delta</th>
<th align="right">rawp</th>
<th align="right">Bonferroni</th>
<th align="right">ABH</th>
</tr>
</thead>
<tbody>
<tr>
<td>NM_024684</td>
<td>AAMDC</td>
<td>chr11</td>
<td>+</td>
<td align="right">77532207</td>
<td align="right">77583398</td>
<td>cmpl</td>
<td>cmpl</td>
<td align="right">2</td>
<td align="right">0</td>
<td align="right">0.079</td>
<td align="right">na</td>
<td>0.924775093657227,0.0331439677875056</td>
<td>na</td>
<td>0.00892857142857145,0.149122807017544</td>
<td>na</td>
<td align="right">0.175073458624837</td>
<td align="right">0.0790256892230577</td>
<td align="right">0.175073458624837</td>
<td align="right">1</td>
<td align="right">0.480780882445856</td>
</tr>
<tr>
<td>NM_015423</td>
<td>AASDHPPT</td>
<td>chr11</td>
<td>+</td>
<td align="right">105948291</td>
<td align="right">105969419</td>
<td>cmpl</td>
<td>cmpl</td>
<td align="right">2</td>
<td align="right">0</td>
<td align="right">0.023</td>
<td align="right">na</td>
<td>0.749258624760841,1</td>
<td>na</td>
<td>0.0384615384615384,0.00769230769230766</td>
<td>na</td>
<td align="right">0.86559726476049</td>
<td align="right">0.023076923076923</td>
<td align="right">0.86559726476049</td>
<td align="right">1</td>
<td align="right">0.873257417545981</td>
</tr>
</tbody>
</table>
<h4 id="single-marker-ase-results">Single marker ASE results</h4>

<p>Single marker ASE results are the raw results from the single marker ASE test.</p>
<ul>
<li><code class="codehilite">chrom</code>: chromosome name</li>
<li><code class="codehilite">pos</code>: genomic start position</li>
<li><code class="codehilite">ref</code>: reference allele genotype</li>
<li><code class="codehilite">mut</code>: non-reference allele genotype</li>
<li><code class="codehilite">cvg_wgs</code>: coverage of the marker from the whole genome sequence (WGS)</li>
<li><code class="codehilite">mut_freq_wgs</code>: non-reference allele fraction in the WGS</li>
<li><code class="codehilite">cvg_rna</code>: coverage of the marker from the RNA-seq</li>
<li><code class="codehilite">mut_freq_rna</code>: non-reference allele fraction in the RNA-seq</li>
<li><code class="codehilite">ref.1</code>: read count of the reference allele in the RNA-seq</li>
<li><code class="codehilite">var</code>: read count of the non-reference allele in the RNA-seq</li>
<li><code class="codehilite">pvalue</code>: p-value from the binomial test</li>
<li><code class="codehilite">delta.abs</code>: absolute difference of the non-reference allele fraction between the WGS and RNA-seq</li>
</ul>
<h5>Example</h5>

<table>
<thead>
<tr>
<th>chrom</th>
<th align="right">pos</th>
<th>ref</th>
<th>mut</th>
<th align="right">cvg_wgs</th>
<th align="right">mut_freq_wgs</th>
<th align="right">cvg_rna</th>
<th align="right">mut_freq_rna</th>
<th align="right">ref.1</th>
<th align="right">var</th>
<th align="right">pvalue</th>
<th align="right">delta.abs</th>
</tr>
</thead>
<tbody>
<tr>
<td>chr11</td>
<td align="right">204147</td>
<td>G</td>
<td>A</td>
<td align="right">36</td>
<td align="right">0.472</td>
<td align="right">85</td>
<td align="right">0.553</td>
<td align="right">38</td>
<td align="right">47</td>
<td align="right">0.385669420119278</td>
<td align="right">0.0529411764705883</td>
</tr>
<tr>
<td>chr11</td>
<td align="right">205198</td>
<td>C</td>
<td>A</td>
<td align="right">23</td>
<td align="right">0.522</td>
<td align="right">83</td>
<td align="right">0.313</td>
<td align="right">57</td>
<td align="right">26</td>
<td align="right">0.000877551780002863</td>
<td align="right">0.186746987951807</td>
</tr>
</tbody>
</table>
                
                  
                
              
              
                


              
            </article>
          </div>
        </div>
      </main>
      
        
<footer class="md-footer">
  
    <div class="md-footer-nav">
      <nav class="md-footer-nav__inner md-grid">
        
          <a href="../warden/" title="WARDEN Differential Expression Analysis" class="md-flex md-footer-nav__link md-footer-nav__link--prev" rel="prev">
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-back md-footer-nav__button"></i>
            </div>
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Previous
                </span>
                WARDEN Differential Expression Analysis
              </span>
            </div>
          </a>
        
        
          <a href="../mutational-spectrum/" title="Mutational Spectrum" class="md-flex md-footer-nav__link md-footer-nav__link--next" rel="next">
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Next
                </span>
                Mutational Spectrum
              </span>
            </div>
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-forward md-footer-nav__button"></i>
            </div>
          </a>
        
      </nav>
    </div>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
          <div class="md-footer-copyright__highlight">
            Copyright &copy; 2017 - 2018 St. Jude Children's Research Hospital
          </div>
        
        
        built from <a href="https://github.com/stjude/sjcloud-docs/tree/76e03fa">76e03fa</a> using
        
        <a href="https://www.mkdocs.org">MkDocs</a>
        and a modified version of
        <a href="https://squidfunk.github.io/mkdocs-material/">
          Material for MkDocs</a>.
      </div>
      
        
  <div class="md-footer-social">
    <link rel="stylesheet" href="../../../assets/fonts/font-awesome.css">
    
      <a href="https://www.facebook.com/stjuderesearch/" class="md-footer-social__link fa fa-facebook"></a>
    
      <a href="https://twitter.com/stjuderesearch?lang=en" class="md-footer-social__link fa fa-twitter"></a>
    
      <a href="https://www.linkedin.com/company/st-jude-childrens-research-hospital" class="md-footer-social__link fa fa-linkedin"></a>
    
      <a href="https://github.com/stjude" class="md-footer-social__link fa fa-github"></a>
    
  </div>

      
    </div>
  </div>
</footer>
      
    </div>
    
      <script src="../../../assets/javascripts/application.js"></script>
      
      <script>app.initialize({version:"0.17.4",url:{base:"../../.."}})</script>
      
    
    
      
    
  </body>
</html>