

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Overview &mdash; St. Jude Cloud Documentation v1.0.0</title>
  

  
  
  
  

  

  
  
    

  

  
    <link rel="stylesheet" href="../../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../../_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="../../../_static/css/custom.css" type="text/css" />
    <link rel="search" title="Search" href="../../../search.html" />
    <link rel="next" title="Pipeline Operation" href="operation.html" />
    <link rel="prev" title="HLA Typing and Neoepitope Prediction" href="index.html" /> 

  
  <script src="../../../_static/js/modernizr.min.js"></script>

</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search">
          

          
            <a href="../../../index.html" class="icon icon-home"> St. Jude Cloud Documentation
          

          
          </a>

          
            
            
              <div class="version">
                latest
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">General Guides</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../../faq.html">Frequently Asked Questions</a></li>
</ul>
<p class="caption"><span class="caption-text">Data Guides</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../data/data-request.html">Make a Data Request</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../data/command-line.html">Getting Started with the Command Line</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../data/upload-data.html">Uploading Data to St. Jude Cloud</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../data/download-data.html">Downloading Data from St. Jude Cloud</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../data/data-transfer-app.html">Data Transfer Application</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../data/dac-protocols.html">Data Access Committee Policies and Protocols</a></li>
</ul>
<p class="caption"><span class="caption-text">Tool Guides</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../pecan-pie/index.html">PeCan PIE</a></li>
<li class="toctree-l1"><a class="reference internal" href="../rapid-rnaseq/index.html">Rapid RNA-Seq Fusion Detection</a></li>
<li class="toctree-l1"><a class="reference internal" href="../chipseq/index.html">ChIP-Seq Peak Calling</a></li>
<li class="toctree-l1"><a class="reference internal" href="../warden/index.html">WARDEN Differential Expression Analysis</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="index.html">HLA Typing and Neoepitope Prediction</a><ul class="current">
<li class="toctree-l2 current"><a class="current reference internal" href="#">Overview</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#inputs">Inputs</a></li>
<li class="toctree-l3"><a class="reference internal" href="#outputs">Outputs</a></li>
<li class="toctree-l3"><a class="reference internal" href="#theory">Theory</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#hla-typing-algorithm">HLA Typing Algorithm</a></li>
<li class="toctree-l4"><a class="reference internal" href="#epitope-prediction-algorithm">Epitope Prediction Algorithm</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="#process">Process</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="operation.html">Pipeline Operation</a></li>
<li class="toctree-l2"><a class="reference internal" href="monitoring.html">Monitoring Run Progress</a></li>
<li class="toctree-l2"><a class="reference internal" href="navigate.html">Navigating Results</a></li>
<li class="toctree-l2"><a class="reference internal" href="results.html">Analysis of Results</a></li>
<li class="toctree-l2"><a class="reference internal" href="faq.html">Frequently Asked Questions</a></li>
</ul>
</li>
</ul>
<p class="caption"><span class="caption-text">Visualization Guides</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../visualizations/bigwig.html">ProteinPaint BigWig Viewer</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../visualizations/fusion-viewer.html">ProteinPaint Fusion Viewer</a></li>
</ul>
<p class="caption"><span class="caption-text">Other Guides</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../internal/index.html">St. Jude Internal Employee Guide</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../../index.html">St. Jude Cloud Documentation</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../../index.html">Docs</a> &raquo;</li>
        
          <li><a href="index.html">HLA Typing and Neoepitope Prediction</a> &raquo;</li>
        
      <li>Overview</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="overview">
<h1>Overview<a class="headerlink" href="#overview" title="Permalink to this headline">¶</a></h1>
<div class="section" id="inputs">
<h2>Inputs<a class="headerlink" href="#inputs" title="Permalink to this headline">¶</a></h2>
<dl class="docutils">
<dt>The HLA typing and neoepitope prediction workflow two required inputs:</dt>
<dd><ol class="first last arabic simple">
<li>FastQs or BAM file and BAM index</li>
<li>Mutation file</li>
</ol>
</dd>
</dl>
<table border="1" class="docutils">
<colgroup>
<col width="16%" />
<col width="10%" />
<col width="55%" />
<col width="18%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Name</th>
<th class="head">Type</th>
<th class="head">Description</th>
<th class="head">Example</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>FastQ files</td>
<td>Input file</td>
<td>Gzipped FastQ files generated by experiment</td>
<td><a href="#id1"><span class="problematic" id="id2">*</span></a>.fastq or <a href="#id3"><span class="problematic" id="id4">*</span></a>.fq</td>
</tr>
<tr class="row-odd"><td>BAM files</td>
<td>Input file</td>
<td>BAM files aligned against Hg19/Hg38 (WGS, WES or RNAseq)</td>
<td><a href="#id5"><span class="problematic" id="id6">*</span></a>.bam</td>
</tr>
<tr class="row-even"><td>BAM index</td>
<td>Input file</td>
<td>BAM index of the BAM file</td>
<td><a href="#id7"><span class="problematic" id="id8">*</span></a>.bai</td>
</tr>
<tr class="row-odd"><td>Mutation file</td>
<td>Input file</td>
<td>Mutation information</td>
<td><a href="#id9"><span class="problematic" id="id10">*</span></a>.txt(tab-delimited)</td>
</tr>
<tr class="row-even"><td>SNV or fusion</td>
<td>Parameter</td>
<td>Specify the mutation file contains SNV or gene fusion</td>
<td>SNV</td>
</tr>
<tr class="row-odd"><td>Peptide size</td>
<td>Parameter</td>
<td>Size of the peptide</td>
<td>9</td>
</tr>
<tr class="row-even"><td>Affinity threshold</td>
<td>Parameter</td>
<td>Affinity cutoff for epitope prediction report</td>
<td>500</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="outputs">
<h2>Outputs<a class="headerlink" href="#outputs" title="Permalink to this headline">¶</a></h2>
<table border="1" class="docutils">
<colgroup>
<col width="32%" />
<col width="68%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Name</th>
<th class="head">Description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Epitope affinity prediction (html)</td>
<td>Epitope affinity. The peptide with affinity &lt; cutoff will be highlighted.</td>
</tr>
<tr class="row-odd"><td>Epitope affinity prediction (xlsx)</td>
<td>Excel tables for the infomation of all epitopes</td>
</tr>
<tr class="row-even"><td>Affinity (raw output)</td>
<td>Epitope affinity</td>
</tr>
<tr class="row-odd"><td>Peptide sequence (raw output)</td>
<td>Peptide sequences in fasta format</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="theory">
<h2>Theory<a class="headerlink" href="#theory" title="Permalink to this headline">¶</a></h2>
<div class="section" id="hla-typing-algorithm">
<h3>HLA Typing Algorithm<a class="headerlink" href="#hla-typing-algorithm" title="Permalink to this headline">¶</a></h3>
<p>The HLA typing algorithm is used to predict the HLA class I alleles. Users
can either provide fastq (paired or single end reads) or a BAM file as input.
When using a BAM file as input, the reads surrounding the HLA loci and unmapped
reads will be extracted. The reads will be fed into Optitype for HLA typing.
The default settings for Optitype are used. The output of the HLA type can be
combined with the our epitope detection algorithm to perform affinity
prediction of neoepitopes (see <a class="reference internal" href="#epitope-pred"><span class="std std-ref">Epitope Prediction Algorithm</span></a>).</p>
</div>
<div class="section" id="epitope-prediction-algorithm">
<span id="epitope-pred"></span><h3>Epitope Prediction Algorithm<a class="headerlink" href="#epitope-prediction-algorithm" title="Permalink to this headline">¶</a></h3>
<p>The stjude-epitope applet is used to extract peptides covering
an array of tiling peptides (size defined by users) overlapping each missense
mutation or gene fusion. Fusion junctions can be identified using RNAseq by
fusion detection tools (e.g. CICERO, Li et al.). NetMHCcons (Karosiene et al.) is
subsequently used to predict affinities of the peptide array for each HLA
receptor in each sample. The neoepitope with affinity lower than the threshold
will be highlighted in output file (default 500 nM).</p>
</div>
</div>
<div class="section" id="process">
<h2>Process<a class="headerlink" href="#process" title="Permalink to this headline">¶</a></h2>
<dl class="docutils">
<dt>HLA typing:</dt>
<dd><dl class="first last docutils">
<dt>If you use <strong>FastQ</strong> files as input:</dt>
<dd><ol class="first last arabic simple">
<li>The input fastqs will be aligned against the Optitype HLA reference sequences using razers3 (see <a class="reference external" href="https://github.com/FRED-2/OptiType">https://github.com/FRED-2/OptiType</a>).</li>
<li>The fished fastqs will be used for HLA typing using Opitype.</li>
</ol>
</dd>
<dt>If you use <strong>BAM</strong> files as input:</dt>
<dd><ol class="first last arabic simple">
<li>The reads falling within the HLA loci and their paralogous loci will be extracted.</li>
<li>The reads unmapped to the human genome will be extracted.</li>
<li>The reads from step 1 and 2 will be combined and deduplicated (in fastq format).</li>
<li>The input fastqs will be aligned against the Optitype HLA reference sequences using razers3 (see <a class="reference external" href="https://github.com/FRED-2/OptiType">https://github.com/FRED-2/OptiType</a>).</li>
<li>The fished fastqs will be used for HLA typing using Opitype.</li>
</ol>
</dd>
</dl>
</dd>
<dt>Neoepitope prediction:</dt>
<dd><ol class="first last arabic simple">
<li>Check the version of the genomic position of the input SNV/fusion file.</li>
<li>lift over the genomic coordinations if the reference genomic position is not Hg19. Currently, the internal genome annotation was based on Hg19 and the genome coordiantes of the mutation files will be adjusted to Hg19 for peptide extraction.</li>
<li>Extract the peptide flanking the mutations.</li>
<li>Run NetMHCcons to obtain the affinity prediction of the peptides.</li>
<li>Produce the affinity report of each peptide.</li>
</ol>
</dd>
</dl>
<p><strong>References</strong></p>
<blockquote>
<div><ol class="arabic simple">
<li>Szolek A, Schubert B, Mohr C, Sturm M, Feldhahn M, Kohlbacher O: OptiType: precision HLA typing from next-generation sequencing data. Bioinformatics 2014, 30:3310-3316.</li>
<li>Karosiene E, Lundegaard C, Lund O, Nielsen M: NetMHCcons: a consensus method for the major histocompatibility complex class I predictions. Immunogenetics 2012, 64:177-186.</li>
</ol>
</div></blockquote>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="operation.html" class="btn btn-neutral float-right" title="Pipeline Operation" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="index.html" class="btn btn-neutral" title="HLA Typing and Neoepitope Prediction" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2016, 2017, 2018 St. Jude Developers.

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  

    <script type="text/javascript">
        var DOCUMENTATION_OPTIONS = {
            URL_ROOT:'../../../',
            VERSION:'latest',
            LANGUAGE:'None',
            COLLAPSE_INDEX:false,
            FILE_SUFFIX:'.html',
            HAS_SOURCE:  true,
            SOURCELINK_SUFFIX: '.txt'
        };
    </script>
      <script type="text/javascript" src="../../../_static/jquery.js"></script>
      <script type="text/javascript" src="../../../_static/underscore.js"></script>
      <script type="text/javascript" src="../../../_static/doctools.js"></script>
      <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

  

  
  
    <script type="text/javascript" src="../../../_static/js/theme.js"></script>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>